2019-11-21 10:21:33,869 [restartedMain] INFO  [,,] com.xiao.emsoft.DemoApplication - Starting DemoApplication on xiaohanfengdeMacBook-Pro.local with PID 12985 (started by xiaohanfeng in /Users/xiaohanfeng/Documents/代码库/账户支付系统/testing)
2019-11-21 10:21:33,874 [restartedMain] INFO  [,,] com.xiao.emsoft.DemoApplication - The following profiles are active: stream,database,cloud
2019-11-21 10:21:33,916 [restartedMain] INFO  [,,] o.s.b.d.e.DevToolsPropertyDefaultsPostProcessor - Devtools property defaults active! Set 'spring.devtools.add-properties' to 'false' to disable
2019-11-21 10:21:33,917 [restartedMain] INFO  [,,] o.s.b.d.e.DevToolsPropertyDefaultsPostProcessor - For additional web related logging consider setting the 'logging.level.web' property to 'DEBUG'
2019-11-21 10:21:35,141 [restartedMain] INFO  [,,] o.s.d.r.c.RepositoryConfigurationDelegate - Bootstrapping Spring Data repositories in DEFAULT mode.
2019-11-21 10:21:35,239 [restartedMain] INFO  [,,] o.s.d.r.c.RepositoryConfigurationDelegate - Finished Spring Data repository scanning in 91ms. Found 1 repository interfaces.
2019-11-21 10:21:35,633 [restartedMain] INFO  [,,] o.s.c.s.PostProcessorRegistrationDelegate$BeanPostProcessorChecker - Bean 'org.springframework.transaction.annotation.ProxyTransactionManagementConfiguration' of type [org.springframework.transaction.annotation.ProxyTransactionManagementConfiguration$$EnhancerBySpringCGLIB$$86702528] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2019-11-21 10:21:36,160 [restartedMain] INFO  [,,] o.s.b.w.e.tomcat.TomcatWebServer - Tomcat initialized with port(s): 8044 (http)
2019-11-21 10:21:36,175 [restartedMain] INFO  [,,] o.a.coyote.http11.Http11NioProtocol - Initializing ProtocolHandler ["http-nio-8044"]
2019-11-21 10:21:36,183 [restartedMain] INFO  [,,] o.a.catalina.core.StandardService - Starting service [Tomcat]
2019-11-21 10:21:36,184 [restartedMain] INFO  [,,] o.a.catalina.core.StandardEngine - Starting Servlet engine: [Apache Tomcat/9.0.19]
2019-11-21 10:21:36,294 [restartedMain] INFO  [,,] o.a.c.c.C.[Tomcat].[localhost].[/] - Initializing Spring embedded WebApplicationContext
2019-11-21 10:21:36,294 [restartedMain] INFO  [,,] o.s.web.context.ContextLoader - Root WebApplicationContext: initialization completed in 2377 ms
2019-11-21 10:21:36,842 [restartedMain] INFO  [,,] com.zaxxer.hikari.HikariDataSource - HikariPool-1 - Starting...
2019-11-21 10:21:42,220 [restartedMain] INFO  [,,] com.zaxxer.hikari.HikariDataSource - HikariPool-1 - Start completed.
2019-11-21 10:21:42,277 [restartedMain] INFO  [,,] o.h.jpa.internal.util.LogHelper - HHH000204: Processing PersistenceUnitInfo [name: default]
2019-11-21 10:21:42,353 [restartedMain] INFO  [,,] org.hibernate.Version - HHH000412: Hibernate Core {5.4.5.Final}
2019-11-21 10:21:42,485 [restartedMain] INFO  [,,] o.h.annotations.common.Version - HCANN000001: Hibernate Commons Annotations {5.1.0.Final}
2019-11-21 10:21:42,603 [restartedMain] INFO  [,,] org.hibernate.dialect.Dialect - HHH000400: Using dialect: org.hibernate.dialect.Oracle10gDialect
2019-11-21 10:21:49,857 [restartedMain] INFO  [,,] o.h.e.t.j.p.i.JtaPlatformInitiator - HHH000490: Using JtaPlatform implementation: [org.hibernate.engine.transaction.jta.platform.internal.NoJtaPlatform]
2019-11-21 10:21:49,865 [restartedMain] INFO  [,,] o.s.o.j.LocalContainerEntityManagerFactoryBean - Initialized JPA EntityManagerFactory for persistence unit 'default'
2019-11-21 10:21:49,907 [restartedMain] INFO  [,,] o.s.b.d.a.OptionalLiveReloadServer - LiveReload server is running on port 35729
2019-11-21 10:21:50,495 [restartedMain] WARN  [,,] o.s.b.a.o.j.JpaBaseConfiguration$JpaWebConfiguration$JpaWebMvcConfiguration - spring.jpa.open-in-view is enabled by default. Therefore, database queries may be performed during view rendering. Explicitly configure spring.jpa.open-in-view to disable this warning
2019-11-21 10:21:50,660 [restartedMain] INFO  [,,] o.s.b.a.e.web.EndpointLinksResolver - Exposing 2 endpoint(s) beneath base path '/actuator'
2019-11-21 10:21:50,731 [restartedMain] INFO  [,,] s.d.s.w.PropertySourcedRequestMappingHandlerMapping - Mapped URL path [/v2/api-docs] onto method [public org.springframework.http.ResponseEntity<springfox.documentation.spring.web.json.Json> springfox.documentation.swagger2.web.Swagger2Controller.getDocumentation(java.lang.String,javax.servlet.http.HttpServletRequest)]
2019-11-21 10:21:50,914 [restartedMain] INFO  [,,] o.s.s.c.ThreadPoolTaskExecutor - Initializing ExecutorService 'applicationTaskExecutor'
2019-11-21 10:21:51,015 [restartedMain] WARN  [,,] o.s.b.a.t.ThymeleafAutoConfiguration$DefaultTemplateResolverConfiguration - Cannot find template location: classpath:/templates/ (please add some templates or check your Thymeleaf configuration)
2019-11-21 10:21:51,498 [restartedMain] WARN  [,,] o.s.b.a.f.FreeMarkerAutoConfiguration - Cannot find template location(s): [classpath:/templates/] (please add some templates, check your FreeMarker configuration, or set spring.freemarker.checkTemplateLocation=false)
2019-11-21 10:21:51,611 [restartedMain] INFO  [,,] s.d.s.w.p.DocumentationPluginsBootstrapper - Context refreshed
2019-11-21 10:21:51,633 [restartedMain] INFO  [,,] s.d.s.w.p.DocumentationPluginsBootstrapper - Found 1 custom documentation plugin(s)
2019-11-21 10:21:51,676 [restartedMain] INFO  [,,] s.d.s.w.s.ApiListingReferenceScanner - Scanning for api listing references
2019-11-21 10:21:51,765 [restartedMain] INFO  [,,] o.a.coyote.http11.Http11NioProtocol - Starting ProtocolHandler ["http-nio-8044"]
2019-11-21 10:21:51,784 [restartedMain] INFO  [,,] o.s.b.w.e.tomcat.TomcatWebServer - Tomcat started on port(s): 8044 (http) with context path ''
2019-11-21 10:21:51,787 [restartedMain] INFO  [,,] com.xiao.emsoft.DemoApplication - Started DemoApplication in 38.437 seconds (JVM running for 44.273)
2019-11-21 10:21:54,183 [RMI TCP Connection(2)-192.168.31.108] INFO  [,,] o.a.c.c.C.[Tomcat].[localhost].[/] - Initializing Spring DispatcherServlet 'dispatcherServlet'
2019-11-21 10:21:54,185 [RMI TCP Connection(2)-192.168.31.108] INFO  [,,] o.s.web.servlet.DispatcherServlet - Initializing Servlet 'dispatcherServlet'
2019-11-21 10:21:54,201 [RMI TCP Connection(2)-192.168.31.108] INFO  [,,] o.s.web.servlet.DispatcherServlet - Completed initialization in 16 ms
2019-11-21 10:23:40,126 [http-nio-8044-exec-5] ERROR [,,] o.a.catalina.connector.CoyoteAdapter - Exception while processing an asynchronous request
java.lang.IllegalStateException: Calling [asyncError()] is not valid for a request with Async state [MUST_DISPATCH]
	at org.apache.coyote.AsyncStateMachine.asyncError(AsyncStateMachine.java:440)
	at org.apache.coyote.AbstractProcessor.action(AbstractProcessor.java:512)
	at org.apache.coyote.Request.action(Request.java:430)
	at org.apache.catalina.core.AsyncContextImpl.setErrorState(AsyncContextImpl.java:401)
	at org.apache.catalina.connector.CoyoteAdapter.asyncDispatch(CoyoteAdapter.java:239)
	at org.apache.coyote.AbstractProcessor.dispatch(AbstractProcessor.java:241)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:53)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:836)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1747)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
2019-11-21 10:23:41,775 [http-nio-8044-exec-7] ERROR [,,] o.a.catalina.connector.CoyoteAdapter - Exception while processing an asynchronous request
java.lang.IllegalStateException: Calling [asyncError()] is not valid for a request with Async state [MUST_DISPATCH]
	at org.apache.coyote.AsyncStateMachine.asyncError(AsyncStateMachine.java:440)
	at org.apache.coyote.AbstractProcessor.action(AbstractProcessor.java:512)
	at org.apache.coyote.Request.action(Request.java:430)
	at org.apache.catalina.core.AsyncContextImpl.setErrorState(AsyncContextImpl.java:401)
	at org.apache.catalina.connector.CoyoteAdapter.asyncDispatch(CoyoteAdapter.java:239)
	at org.apache.coyote.AbstractProcessor.dispatch(AbstractProcessor.java:241)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:53)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:836)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1747)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
2019-11-21 10:40:07,846 [Thread-70] INFO  [,,] o.s.s.c.ThreadPoolTaskExecutor - Shutting down ExecutorService 'applicationTaskExecutor'
2019-11-21 10:40:07,849 [Thread-70] INFO  [,,] o.s.o.j.LocalContainerEntityManagerFactoryBean - Closing JPA EntityManagerFactory for persistence unit 'default'
2019-11-21 10:40:07,852 [Thread-70] INFO  [,,] com.zaxxer.hikari.HikariDataSource - HikariPool-1 - Shutdown initiated...
2019-11-21 10:40:07,893 [Thread-70] INFO  [,,] com.zaxxer.hikari.HikariDataSource - HikariPool-1 - Shutdown completed.
2019-11-21 10:40:40,897 [restartedMain] INFO  [,,] com.xiao.emsoft.DemoApplication - Starting DemoApplication on xiaohanfengdeMacBook-Pro.local with PID 13049 (started by xiaohanfeng in /Users/xiaohanfeng/Documents/代码库/账户支付系统/testing)
2019-11-21 10:40:40,899 [restartedMain] INFO  [,,] com.xiao.emsoft.DemoApplication - The following profiles are active: stream,database,cloud
2019-11-21 10:40:40,937 [restartedMain] INFO  [,,] o.s.b.d.e.DevToolsPropertyDefaultsPostProcessor - Devtools property defaults active! Set 'spring.devtools.add-properties' to 'false' to disable
2019-11-21 10:40:40,937 [restartedMain] INFO  [,,] o.s.b.d.e.DevToolsPropertyDefaultsPostProcessor - For additional web related logging consider setting the 'logging.level.web' property to 'DEBUG'
2019-11-21 10:40:41,794 [restartedMain] INFO  [,,] o.s.d.r.c.RepositoryConfigurationDelegate - Bootstrapping Spring Data repositories in DEFAULT mode.
2019-11-21 10:40:41,870 [restartedMain] INFO  [,,] o.s.d.r.c.RepositoryConfigurationDelegate - Finished Spring Data repository scanning in 72ms. Found 1 repository interfaces.
2019-11-21 10:40:42,192 [restartedMain] INFO  [,,] o.s.c.s.PostProcessorRegistrationDelegate$BeanPostProcessorChecker - Bean 'org.springframework.transaction.annotation.ProxyTransactionManagementConfiguration' of type [org.springframework.transaction.annotation.ProxyTransactionManagementConfiguration$$EnhancerBySpringCGLIB$$9d239409] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2019-11-21 10:40:42,589 [restartedMain] INFO  [,,] o.s.b.w.e.tomcat.TomcatWebServer - Tomcat initialized with port(s): 8045 (http)
2019-11-21 10:40:42,598 [restartedMain] INFO  [,,] o.a.coyote.http11.Http11NioProtocol - Initializing ProtocolHandler ["http-nio-8045"]
2019-11-21 10:40:42,604 [restartedMain] INFO  [,,] o.a.catalina.core.StandardService - Starting service [Tomcat]
2019-11-21 10:40:42,604 [restartedMain] INFO  [,,] o.a.catalina.core.StandardEngine - Starting Servlet engine: [Apache Tomcat/9.0.19]
2019-11-21 10:40:42,685 [restartedMain] INFO  [,,] o.a.c.c.C.[Tomcat].[localhost].[/] - Initializing Spring embedded WebApplicationContext
2019-11-21 10:40:42,685 [restartedMain] INFO  [,,] o.s.web.context.ContextLoader - Root WebApplicationContext: initialization completed in 1748 ms
2019-11-21 10:40:43,110 [restartedMain] INFO  [,,] com.zaxxer.hikari.HikariDataSource - HikariPool-1 - Starting...
2019-11-21 10:40:48,419 [restartedMain] INFO  [,,] com.zaxxer.hikari.HikariDataSource - HikariPool-1 - Start completed.
2019-11-21 10:40:48,458 [restartedMain] INFO  [,,] o.h.jpa.internal.util.LogHelper - HHH000204: Processing PersistenceUnitInfo [name: default]
2019-11-21 10:40:48,505 [restartedMain] INFO  [,,] org.hibernate.Version - HHH000412: Hibernate Core {5.4.5.Final}
2019-11-21 10:40:48,591 [restartedMain] INFO  [,,] o.h.annotations.common.Version - HCANN000001: Hibernate Commons Annotations {5.1.0.Final}
2019-11-21 10:40:48,695 [restartedMain] INFO  [,,] org.hibernate.dialect.Dialect - HHH000400: Using dialect: org.hibernate.dialect.Oracle10gDialect
2019-11-21 10:40:52,385 [restartedMain] INFO  [,,] o.h.e.t.j.p.i.JtaPlatformInitiator - HHH000490: Using JtaPlatform implementation: [org.hibernate.engine.transaction.jta.platform.internal.NoJtaPlatform]
2019-11-21 10:40:52,395 [restartedMain] INFO  [,,] o.s.o.j.LocalContainerEntityManagerFactoryBean - Initialized JPA EntityManagerFactory for persistence unit 'default'
2019-11-21 10:40:52,434 [restartedMain] INFO  [,,] o.s.b.d.a.OptionalLiveReloadServer - LiveReload server is running on port 35729
2019-11-21 10:40:52,913 [restartedMain] WARN  [,,] o.s.b.a.o.j.JpaBaseConfiguration$JpaWebConfiguration$JpaWebMvcConfiguration - spring.jpa.open-in-view is enabled by default. Therefore, database queries may be performed during view rendering. Explicitly configure spring.jpa.open-in-view to disable this warning
2019-11-21 10:40:53,037 [restartedMain] INFO  [,,] o.s.b.a.e.web.EndpointLinksResolver - Exposing 2 endpoint(s) beneath base path '/actuator'
2019-11-21 10:40:53,094 [restartedMain] INFO  [,,] s.d.s.w.PropertySourcedRequestMappingHandlerMapping - Mapped URL path [/v2/api-docs] onto method [public org.springframework.http.ResponseEntity<springfox.documentation.spring.web.json.Json> springfox.documentation.swagger2.web.Swagger2Controller.getDocumentation(java.lang.String,javax.servlet.http.HttpServletRequest)]
2019-11-21 10:40:53,243 [restartedMain] INFO  [,,] o.s.s.c.ThreadPoolTaskExecutor - Initializing ExecutorService 'applicationTaskExecutor'
2019-11-21 10:40:53,326 [restartedMain] WARN  [,,] o.s.b.a.t.ThymeleafAutoConfiguration$DefaultTemplateResolverConfiguration - Cannot find template location: classpath:/templates/ (please add some templates or check your Thymeleaf configuration)
2019-11-21 10:40:53,671 [restartedMain] WARN  [,,] o.s.b.a.f.FreeMarkerAutoConfiguration - Cannot find template location(s): [classpath:/templates/] (please add some templates, check your FreeMarker configuration, or set spring.freemarker.checkTemplateLocation=false)
2019-11-21 10:40:53,761 [restartedMain] INFO  [,,] s.d.s.w.p.DocumentationPluginsBootstrapper - Context refreshed
2019-11-21 10:40:53,776 [restartedMain] INFO  [,,] s.d.s.w.p.DocumentationPluginsBootstrapper - Found 1 custom documentation plugin(s)
2019-11-21 10:40:53,808 [restartedMain] INFO  [,,] s.d.s.w.s.ApiListingReferenceScanner - Scanning for api listing references
2019-11-21 10:40:53,868 [restartedMain] INFO  [,,] o.a.coyote.http11.Http11NioProtocol - Starting ProtocolHandler ["http-nio-8045"]
2019-11-21 10:40:53,886 [restartedMain] INFO  [,,] o.s.b.w.e.tomcat.TomcatWebServer - Tomcat started on port(s): 8045 (http) with context path ''
2019-11-21 10:40:53,889 [restartedMain] INFO  [,,] com.xiao.emsoft.DemoApplication - Started DemoApplication in 33.372 seconds (JVM running for 39.097)
2019-11-21 10:40:53,981 [RMI TCP Connection(2)-192.168.31.108] INFO  [,,] o.a.c.c.C.[Tomcat].[localhost].[/] - Initializing Spring DispatcherServlet 'dispatcherServlet'
2019-11-21 10:40:53,982 [RMI TCP Connection(2)-192.168.31.108] INFO  [,,] o.s.web.servlet.DispatcherServlet - Initializing Servlet 'dispatcherServlet'
2019-11-21 10:40:53,991 [RMI TCP Connection(2)-192.168.31.108] INFO  [,,] o.s.web.servlet.DispatcherServlet - Completed initialization in 9 ms
2019-11-21 10:44:53,311 [http-nio-8045-exec-2] ERROR [,,] o.a.catalina.connector.CoyoteAdapter - Exception while processing an asynchronous request
java.lang.IllegalStateException: Calling [asyncError()] is not valid for a request with Async state [MUST_DISPATCH]
	at org.apache.coyote.AsyncStateMachine.asyncError(AsyncStateMachine.java:440)
	at org.apache.coyote.AbstractProcessor.action(AbstractProcessor.java:512)
	at org.apache.coyote.Request.action(Request.java:430)
	at org.apache.catalina.core.AsyncContextImpl.setErrorState(AsyncContextImpl.java:401)
	at org.apache.catalina.connector.CoyoteAdapter.asyncDispatch(CoyoteAdapter.java:239)
	at org.apache.coyote.AbstractProcessor.dispatch(AbstractProcessor.java:241)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:53)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:836)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1747)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
2019-11-21 10:44:57,027 [http-nio-8045-exec-6] ERROR [,,] o.a.catalina.connector.CoyoteAdapter - Exception while processing an asynchronous request
java.lang.IllegalStateException: Calling [asyncError()] is not valid for a request with Async state [MUST_DISPATCH]
	at org.apache.coyote.AsyncStateMachine.asyncError(AsyncStateMachine.java:440)
	at org.apache.coyote.AbstractProcessor.action(AbstractProcessor.java:512)
	at org.apache.coyote.Request.action(Request.java:430)
	at org.apache.catalina.core.AsyncContextImpl.setErrorState(AsyncContextImpl.java:401)
	at org.apache.catalina.connector.CoyoteAdapter.asyncDispatch(CoyoteAdapter.java:239)
	at org.apache.coyote.AbstractProcessor.dispatch(AbstractProcessor.java:241)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:53)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:836)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1747)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
2019-11-21 10:44:57,592 [http-nio-8045-exec-5] ERROR [,,] o.a.catalina.connector.CoyoteAdapter - Exception while processing an asynchronous request
java.lang.IllegalStateException: Calling [asyncError()] is not valid for a request with Async state [MUST_DISPATCH]
	at org.apache.coyote.AsyncStateMachine.asyncError(AsyncStateMachine.java:440)
	at org.apache.coyote.AbstractProcessor.action(AbstractProcessor.java:512)
	at org.apache.coyote.Request.action(Request.java:430)
	at org.apache.catalina.core.AsyncContextImpl.setErrorState(AsyncContextImpl.java:401)
	at org.apache.catalina.connector.CoyoteAdapter.asyncDispatch(CoyoteAdapter.java:239)
	at org.apache.coyote.AbstractProcessor.dispatch(AbstractProcessor.java:241)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:53)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:836)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1747)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
2019-11-21 10:44:57,759 [http-nio-8045-exec-7] ERROR [,,] o.a.catalina.connector.CoyoteAdapter - Exception while processing an asynchronous request
java.lang.IllegalStateException: Calling [asyncError()] is not valid for a request with Async state [MUST_DISPATCH]
	at org.apache.coyote.AsyncStateMachine.asyncError(AsyncStateMachine.java:440)
	at org.apache.coyote.AbstractProcessor.action(AbstractProcessor.java:512)
	at org.apache.coyote.Request.action(Request.java:430)
	at org.apache.catalina.core.AsyncContextImpl.setErrorState(AsyncContextImpl.java:401)
	at org.apache.catalina.connector.CoyoteAdapter.asyncDispatch(CoyoteAdapter.java:239)
	at org.apache.coyote.AbstractProcessor.dispatch(AbstractProcessor.java:241)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:53)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:836)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1747)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
2019-11-21 10:45:09,996 [http-nio-8045-exec-9] ERROR [,,] o.a.catalina.connector.CoyoteAdapter - Exception while processing an asynchronous request
java.lang.IllegalStateException: Calling [asyncError()] is not valid for a request with Async state [MUST_DISPATCH]
	at org.apache.coyote.AsyncStateMachine.asyncError(AsyncStateMachine.java:440)
	at org.apache.coyote.AbstractProcessor.action(AbstractProcessor.java:512)
	at org.apache.coyote.Request.action(Request.java:430)
	at org.apache.catalina.core.AsyncContextImpl.setErrorState(AsyncContextImpl.java:401)
	at org.apache.catalina.connector.CoyoteAdapter.asyncDispatch(CoyoteAdapter.java:239)
	at org.apache.coyote.AbstractProcessor.dispatch(AbstractProcessor.java:241)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:53)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:836)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1747)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
2019-11-21 10:59:47,954 [http-nio-8045-exec-8] ERROR [,,] o.a.catalina.connector.CoyoteAdapter - Exception while processing an asynchronous request
java.lang.IllegalStateException: Calling [asyncError()] is not valid for a request with Async state [MUST_DISPATCH]
	at org.apache.coyote.AsyncStateMachine.asyncError(AsyncStateMachine.java:440)
	at org.apache.coyote.AbstractProcessor.action(AbstractProcessor.java:512)
	at org.apache.coyote.Request.action(Request.java:430)
	at org.apache.catalina.core.AsyncContextImpl.setErrorState(AsyncContextImpl.java:401)
	at org.apache.catalina.connector.CoyoteAdapter.asyncDispatch(CoyoteAdapter.java:239)
	at org.apache.coyote.AbstractProcessor.dispatch(AbstractProcessor.java:241)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:53)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:836)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1747)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
2019-11-21 10:59:56,844 [http-nio-8045-exec-1] ERROR [,,] o.a.catalina.connector.CoyoteAdapter - Exception while processing an asynchronous request
java.lang.IllegalStateException: Calling [asyncError()] is not valid for a request with Async state [MUST_DISPATCH]
	at org.apache.coyote.AsyncStateMachine.asyncError(AsyncStateMachine.java:440)
	at org.apache.coyote.AbstractProcessor.action(AbstractProcessor.java:512)
	at org.apache.coyote.Request.action(Request.java:430)
	at org.apache.catalina.core.AsyncContextImpl.setErrorState(AsyncContextImpl.java:401)
	at org.apache.catalina.connector.CoyoteAdapter.asyncDispatch(CoyoteAdapter.java:239)
	at org.apache.coyote.AbstractProcessor.dispatch(AbstractProcessor.java:241)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:53)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:836)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1747)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
2019-11-21 10:59:57,819 [http-nio-8045-exec-4] ERROR [,,] o.a.catalina.connector.CoyoteAdapter - Exception while processing an asynchronous request
java.lang.IllegalStateException: Calling [asyncError()] is not valid for a request with Async state [MUST_DISPATCH]
	at org.apache.coyote.AsyncStateMachine.asyncError(AsyncStateMachine.java:440)
	at org.apache.coyote.AbstractProcessor.action(AbstractProcessor.java:512)
	at org.apache.coyote.Request.action(Request.java:430)
	at org.apache.catalina.core.AsyncContextImpl.setErrorState(AsyncContextImpl.java:401)
	at org.apache.catalina.connector.CoyoteAdapter.asyncDispatch(CoyoteAdapter.java:239)
	at org.apache.coyote.AbstractProcessor.dispatch(AbstractProcessor.java:241)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:53)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:836)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1747)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
2019-11-21 10:59:58,228 [http-nio-8045-exec-3] ERROR [,,] o.a.catalina.connector.CoyoteAdapter - Exception while processing an asynchronous request
java.lang.IllegalStateException: Calling [asyncError()] is not valid for a request with Async state [MUST_DISPATCH]
	at org.apache.coyote.AsyncStateMachine.asyncError(AsyncStateMachine.java:440)
	at org.apache.coyote.AbstractProcessor.action(AbstractProcessor.java:512)
	at org.apache.coyote.Request.action(Request.java:430)
	at org.apache.catalina.core.AsyncContextImpl.setErrorState(AsyncContextImpl.java:401)
	at org.apache.catalina.connector.CoyoteAdapter.asyncDispatch(CoyoteAdapter.java:239)
	at org.apache.coyote.AbstractProcessor.dispatch(AbstractProcessor.java:241)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:53)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:836)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1747)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
2019-11-21 10:59:58,392 [http-nio-8045-exec-2] ERROR [,,] o.a.catalina.connector.CoyoteAdapter - Exception while processing an asynchronous request
java.lang.IllegalStateException: Calling [asyncError()] is not valid for a request with Async state [MUST_DISPATCH]
	at org.apache.coyote.AsyncStateMachine.asyncError(AsyncStateMachine.java:440)
	at org.apache.coyote.AbstractProcessor.action(AbstractProcessor.java:512)
	at org.apache.coyote.Request.action(Request.java:430)
	at org.apache.catalina.core.AsyncContextImpl.setErrorState(AsyncContextImpl.java:401)
	at org.apache.catalina.connector.CoyoteAdapter.asyncDispatch(CoyoteAdapter.java:239)
	at org.apache.coyote.AbstractProcessor.dispatch(AbstractProcessor.java:241)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:53)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:836)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1747)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
2019-11-21 10:59:58,546 [http-nio-8045-exec-6] ERROR [,,] o.a.catalina.connector.CoyoteAdapter - Exception while processing an asynchronous request
java.lang.IllegalStateException: Calling [asyncError()] is not valid for a request with Async state [MUST_DISPATCH]
	at org.apache.coyote.AsyncStateMachine.asyncError(AsyncStateMachine.java:440)
	at org.apache.coyote.AbstractProcessor.action(AbstractProcessor.java:512)
	at org.apache.coyote.Request.action(Request.java:430)
	at org.apache.catalina.core.AsyncContextImpl.setErrorState(AsyncContextImpl.java:401)
	at org.apache.catalina.connector.CoyoteAdapter.asyncDispatch(CoyoteAdapter.java:239)
	at org.apache.coyote.AbstractProcessor.dispatch(AbstractProcessor.java:241)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:53)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:836)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1747)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
2019-11-21 10:59:58,733 [http-nio-8045-exec-5] ERROR [,,] o.a.catalina.connector.CoyoteAdapter - Exception while processing an asynchronous request
java.lang.IllegalStateException: Calling [asyncError()] is not valid for a request with Async state [MUST_DISPATCH]
	at org.apache.coyote.AsyncStateMachine.asyncError(AsyncStateMachine.java:440)
	at org.apache.coyote.AbstractProcessor.action(AbstractProcessor.java:512)
	at org.apache.coyote.Request.action(Request.java:430)
	at org.apache.catalina.core.AsyncContextImpl.setErrorState(AsyncContextImpl.java:401)
	at org.apache.catalina.connector.CoyoteAdapter.asyncDispatch(CoyoteAdapter.java:239)
	at org.apache.coyote.AbstractProcessor.dispatch(AbstractProcessor.java:241)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:53)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:836)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1747)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
2019-11-21 10:59:58,915 [http-nio-8045-exec-7] ERROR [,,] o.a.catalina.connector.CoyoteAdapter - Exception while processing an asynchronous request
java.lang.IllegalStateException: Calling [asyncError()] is not valid for a request with Async state [MUST_DISPATCH]
	at org.apache.coyote.AsyncStateMachine.asyncError(AsyncStateMachine.java:440)
	at org.apache.coyote.AbstractProcessor.action(AbstractProcessor.java:512)
	at org.apache.coyote.Request.action(Request.java:430)
	at org.apache.catalina.core.AsyncContextImpl.setErrorState(AsyncContextImpl.java:401)
	at org.apache.catalina.connector.CoyoteAdapter.asyncDispatch(CoyoteAdapter.java:239)
	at org.apache.coyote.AbstractProcessor.dispatch(AbstractProcessor.java:241)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:53)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:836)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1747)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
2019-11-21 11:00:29,115 [http-nio-8045-exec-3] ERROR [,,] o.a.catalina.connector.CoyoteAdapter - Exception while processing an asynchronous request
java.lang.IllegalStateException: Calling [asyncError()] is not valid for a request with Async state [MUST_DISPATCH]
	at org.apache.coyote.AsyncStateMachine.asyncError(AsyncStateMachine.java:440)
	at org.apache.coyote.AbstractProcessor.action(AbstractProcessor.java:512)
	at org.apache.coyote.Request.action(Request.java:430)
	at org.apache.catalina.core.AsyncContextImpl.setErrorState(AsyncContextImpl.java:401)
	at org.apache.catalina.connector.CoyoteAdapter.asyncDispatch(CoyoteAdapter.java:239)
	at org.apache.coyote.AbstractProcessor.dispatch(AbstractProcessor.java:241)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:53)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:836)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1747)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
2019-11-21 11:00:36,800 [http-nio-8045-exec-2] ERROR [,,] o.a.catalina.connector.CoyoteAdapter - Exception while processing an asynchronous request
java.lang.IllegalStateException: Calling [asyncError()] is not valid for a request with Async state [MUST_DISPATCH]
	at org.apache.coyote.AsyncStateMachine.asyncError(AsyncStateMachine.java:440)
	at org.apache.coyote.AbstractProcessor.action(AbstractProcessor.java:512)
	at org.apache.coyote.Request.action(Request.java:430)
	at org.apache.catalina.core.AsyncContextImpl.setErrorState(AsyncContextImpl.java:401)
	at org.apache.catalina.connector.CoyoteAdapter.asyncDispatch(CoyoteAdapter.java:239)
	at org.apache.coyote.AbstractProcessor.dispatch(AbstractProcessor.java:241)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:53)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:836)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1747)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
2019-11-21 11:00:37,891 [http-nio-8045-exec-6] ERROR [,,] o.a.catalina.connector.CoyoteAdapter - Exception while processing an asynchronous request
java.lang.IllegalStateException: Calling [asyncError()] is not valid for a request with Async state [MUST_DISPATCH]
	at org.apache.coyote.AsyncStateMachine.asyncError(AsyncStateMachine.java:440)
	at org.apache.coyote.AbstractProcessor.action(AbstractProcessor.java:512)
	at org.apache.coyote.Request.action(Request.java:430)
	at org.apache.catalina.core.AsyncContextImpl.setErrorState(AsyncContextImpl.java:401)
	at org.apache.catalina.connector.CoyoteAdapter.asyncDispatch(CoyoteAdapter.java:239)
	at org.apache.coyote.AbstractProcessor.dispatch(AbstractProcessor.java:241)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:53)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:836)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1747)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
2019-11-21 11:00:38,049 [http-nio-8045-exec-5] ERROR [,,] o.a.catalina.connector.CoyoteAdapter - Exception while processing an asynchronous request
java.lang.IllegalStateException: Calling [asyncError()] is not valid for a request with Async state [MUST_DISPATCH]
	at org.apache.coyote.AsyncStateMachine.asyncError(AsyncStateMachine.java:440)
	at org.apache.coyote.AbstractProcessor.action(AbstractProcessor.java:512)
	at org.apache.coyote.Request.action(Request.java:430)
	at org.apache.catalina.core.AsyncContextImpl.setErrorState(AsyncContextImpl.java:401)
	at org.apache.catalina.connector.CoyoteAdapter.asyncDispatch(CoyoteAdapter.java:239)
	at org.apache.coyote.AbstractProcessor.dispatch(AbstractProcessor.java:241)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:53)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:836)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1747)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
2019-11-21 11:00:38,191 [http-nio-8045-exec-7] ERROR [,,] o.a.catalina.connector.CoyoteAdapter - Exception while processing an asynchronous request
java.lang.IllegalStateException: Calling [asyncError()] is not valid for a request with Async state [MUST_DISPATCH]
	at org.apache.coyote.AsyncStateMachine.asyncError(AsyncStateMachine.java:440)
	at org.apache.coyote.AbstractProcessor.action(AbstractProcessor.java:512)
	at org.apache.coyote.Request.action(Request.java:430)
	at org.apache.catalina.core.AsyncContextImpl.setErrorState(AsyncContextImpl.java:401)
	at org.apache.catalina.connector.CoyoteAdapter.asyncDispatch(CoyoteAdapter.java:239)
	at org.apache.coyote.AbstractProcessor.dispatch(AbstractProcessor.java:241)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:53)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:836)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1747)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
2019-11-21 11:00:41,506 [http-nio-8045-exec-9] ERROR [,,] o.a.catalina.connector.CoyoteAdapter - Exception while processing an asynchronous request
java.lang.IllegalStateException: Calling [asyncError()] is not valid for a request with Async state [MUST_DISPATCH]
	at org.apache.coyote.AsyncStateMachine.asyncError(AsyncStateMachine.java:440)
	at org.apache.coyote.AbstractProcessor.action(AbstractProcessor.java:512)
	at org.apache.coyote.Request.action(Request.java:430)
	at org.apache.catalina.core.AsyncContextImpl.setErrorState(AsyncContextImpl.java:401)
	at org.apache.catalina.connector.CoyoteAdapter.asyncDispatch(CoyoteAdapter.java:239)
	at org.apache.coyote.AbstractProcessor.dispatch(AbstractProcessor.java:241)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:53)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:836)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1747)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
2019-11-21 11:00:42,061 [http-nio-8045-exec-10] ERROR [,,] o.a.catalina.connector.CoyoteAdapter - Exception while processing an asynchronous request
java.lang.IllegalStateException: Calling [asyncError()] is not valid for a request with Async state [MUST_DISPATCH]
	at org.apache.coyote.AsyncStateMachine.asyncError(AsyncStateMachine.java:440)
	at org.apache.coyote.AbstractProcessor.action(AbstractProcessor.java:512)
	at org.apache.coyote.Request.action(Request.java:430)
	at org.apache.catalina.core.AsyncContextImpl.setErrorState(AsyncContextImpl.java:401)
	at org.apache.catalina.connector.CoyoteAdapter.asyncDispatch(CoyoteAdapter.java:239)
	at org.apache.coyote.AbstractProcessor.dispatch(AbstractProcessor.java:241)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:53)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:836)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1747)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
2019-11-21 11:00:42,245 [http-nio-8045-exec-8] ERROR [,,] o.a.catalina.connector.CoyoteAdapter - Exception while processing an asynchronous request
java.lang.IllegalStateException: Calling [asyncError()] is not valid for a request with Async state [MUST_DISPATCH]
	at org.apache.coyote.AsyncStateMachine.asyncError(AsyncStateMachine.java:440)
	at org.apache.coyote.AbstractProcessor.action(AbstractProcessor.java:512)
	at org.apache.coyote.Request.action(Request.java:430)
	at org.apache.catalina.core.AsyncContextImpl.setErrorState(AsyncContextImpl.java:401)
	at org.apache.catalina.connector.CoyoteAdapter.asyncDispatch(CoyoteAdapter.java:239)
	at org.apache.coyote.AbstractProcessor.dispatch(AbstractProcessor.java:241)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:53)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:836)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1747)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
2019-11-21 11:00:42,414 [http-nio-8045-exec-1] ERROR [,,] o.a.catalina.connector.CoyoteAdapter - Exception while processing an asynchronous request
java.lang.IllegalStateException: Calling [asyncError()] is not valid for a request with Async state [MUST_DISPATCH]
	at org.apache.coyote.AsyncStateMachine.asyncError(AsyncStateMachine.java:440)
	at org.apache.coyote.AbstractProcessor.action(AbstractProcessor.java:512)
	at org.apache.coyote.Request.action(Request.java:430)
	at org.apache.catalina.core.AsyncContextImpl.setErrorState(AsyncContextImpl.java:401)
	at org.apache.catalina.connector.CoyoteAdapter.asyncDispatch(CoyoteAdapter.java:239)
	at org.apache.coyote.AbstractProcessor.dispatch(AbstractProcessor.java:241)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:53)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:836)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1747)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
2019-11-21 11:00:42,583 [http-nio-8045-exec-4] ERROR [,,] o.a.catalina.connector.CoyoteAdapter - Exception while processing an asynchronous request
java.lang.IllegalStateException: Calling [asyncError()] is not valid for a request with Async state [MUST_DISPATCH]
	at org.apache.coyote.AsyncStateMachine.asyncError(AsyncStateMachine.java:440)
	at org.apache.coyote.AbstractProcessor.action(AbstractProcessor.java:512)
	at org.apache.coyote.Request.action(Request.java:430)
	at org.apache.catalina.core.AsyncContextImpl.setErrorState(AsyncContextImpl.java:401)
	at org.apache.catalina.connector.CoyoteAdapter.asyncDispatch(CoyoteAdapter.java:239)
	at org.apache.coyote.AbstractProcessor.dispatch(AbstractProcessor.java:241)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:53)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:836)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1747)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
2019-11-21 11:00:42,769 [http-nio-8045-exec-3] ERROR [,,] o.a.catalina.connector.CoyoteAdapter - Exception while processing an asynchronous request
java.lang.IllegalStateException: Calling [asyncError()] is not valid for a request with Async state [MUST_DISPATCH]
	at org.apache.coyote.AsyncStateMachine.asyncError(AsyncStateMachine.java:440)
	at org.apache.coyote.AbstractProcessor.action(AbstractProcessor.java:512)
	at org.apache.coyote.Request.action(Request.java:430)
	at org.apache.catalina.core.AsyncContextImpl.setErrorState(AsyncContextImpl.java:401)
	at org.apache.catalina.connector.CoyoteAdapter.asyncDispatch(CoyoteAdapter.java:239)
	at org.apache.coyote.AbstractProcessor.dispatch(AbstractProcessor.java:241)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:53)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:836)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1747)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
2019-11-21 11:00:42,935 [http-nio-8045-exec-2] ERROR [,,] o.a.catalina.connector.CoyoteAdapter - Exception while processing an asynchronous request
java.lang.IllegalStateException: Calling [asyncError()] is not valid for a request with Async state [MUST_DISPATCH]
	at org.apache.coyote.AsyncStateMachine.asyncError(AsyncStateMachine.java:440)
	at org.apache.coyote.AbstractProcessor.action(AbstractProcessor.java:512)
	at org.apache.coyote.Request.action(Request.java:430)
	at org.apache.catalina.core.AsyncContextImpl.setErrorState(AsyncContextImpl.java:401)
	at org.apache.catalina.connector.CoyoteAdapter.asyncDispatch(CoyoteAdapter.java:239)
	at org.apache.coyote.AbstractProcessor.dispatch(AbstractProcessor.java:241)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:53)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:836)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1747)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
2019-11-21 11:00:43,244 [http-nio-8045-exec-6] ERROR [,,] o.a.catalina.connector.CoyoteAdapter - Exception while processing an asynchronous request
java.lang.IllegalStateException: Calling [asyncError()] is not valid for a request with Async state [MUST_DISPATCH]
	at org.apache.coyote.AsyncStateMachine.asyncError(AsyncStateMachine.java:440)
	at org.apache.coyote.AbstractProcessor.action(AbstractProcessor.java:512)
	at org.apache.coyote.Request.action(Request.java:430)
	at org.apache.catalina.core.AsyncContextImpl.setErrorState(AsyncContextImpl.java:401)
	at org.apache.catalina.connector.CoyoteAdapter.asyncDispatch(CoyoteAdapter.java:239)
	at org.apache.coyote.AbstractProcessor.dispatch(AbstractProcessor.java:241)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:53)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:836)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1747)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
2019-11-21 11:00:43,431 [http-nio-8045-exec-5] ERROR [,,] o.a.catalina.connector.CoyoteAdapter - Exception while processing an asynchronous request
java.lang.IllegalStateException: Calling [asyncError()] is not valid for a request with Async state [MUST_DISPATCH]
	at org.apache.coyote.AsyncStateMachine.asyncError(AsyncStateMachine.java:440)
	at org.apache.coyote.AbstractProcessor.action(AbstractProcessor.java:512)
	at org.apache.coyote.Request.action(Request.java:430)
	at org.apache.catalina.core.AsyncContextImpl.setErrorState(AsyncContextImpl.java:401)
	at org.apache.catalina.connector.CoyoteAdapter.asyncDispatch(CoyoteAdapter.java:239)
	at org.apache.coyote.AbstractProcessor.dispatch(AbstractProcessor.java:241)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:53)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:836)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1747)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
2019-11-21 11:00:43,606 [http-nio-8045-exec-7] ERROR [,,] o.a.catalina.connector.CoyoteAdapter - Exception while processing an asynchronous request
java.lang.IllegalStateException: Calling [asyncError()] is not valid for a request with Async state [MUST_DISPATCH]
	at org.apache.coyote.AsyncStateMachine.asyncError(AsyncStateMachine.java:440)
	at org.apache.coyote.AbstractProcessor.action(AbstractProcessor.java:512)
	at org.apache.coyote.Request.action(Request.java:430)
	at org.apache.catalina.core.AsyncContextImpl.setErrorState(AsyncContextImpl.java:401)
	at org.apache.catalina.connector.CoyoteAdapter.asyncDispatch(CoyoteAdapter.java:239)
	at org.apache.coyote.AbstractProcessor.dispatch(AbstractProcessor.java:241)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:53)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:836)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1747)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
2019-11-21 11:05:43,801 [http-nio-8045-exec-1] ERROR [,,] o.a.catalina.connector.CoyoteAdapter - Exception while processing an asynchronous request
java.lang.IllegalStateException: Calling [asyncError()] is not valid for a request with Async state [MUST_DISPATCH]
	at org.apache.coyote.AsyncStateMachine.asyncError(AsyncStateMachine.java:440)
	at org.apache.coyote.AbstractProcessor.action(AbstractProcessor.java:512)
	at org.apache.coyote.Request.action(Request.java:430)
	at org.apache.catalina.core.AsyncContextImpl.setErrorState(AsyncContextImpl.java:401)
	at org.apache.catalina.connector.CoyoteAdapter.asyncDispatch(CoyoteAdapter.java:239)
	at org.apache.coyote.AbstractProcessor.dispatch(AbstractProcessor.java:241)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:53)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:836)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1747)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
2019-11-21 11:10:31,742 [HikariPool-1 housekeeper] WARN  [,,] com.zaxxer.hikari.pool.HikariPool - HikariPool-1 - Thread starvation or clock leap detected (housekeeper delta=5m13s38ms).
2019-11-21 11:12:21,937 [http-nio-8045-exec-4] ERROR [,,] o.a.catalina.connector.CoyoteAdapter - Exception while processing an asynchronous request
java.lang.IllegalStateException: Calling [asyncError()] is not valid for a request with Async state [MUST_DISPATCH]
	at org.apache.coyote.AsyncStateMachine.asyncError(AsyncStateMachine.java:440)
	at org.apache.coyote.AbstractProcessor.action(AbstractProcessor.java:512)
	at org.apache.coyote.Request.action(Request.java:430)
	at org.apache.catalina.core.AsyncContextImpl.setErrorState(AsyncContextImpl.java:401)
	at org.apache.catalina.connector.CoyoteAdapter.asyncDispatch(CoyoteAdapter.java:239)
	at org.apache.coyote.AbstractProcessor.dispatch(AbstractProcessor.java:241)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:53)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:836)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1747)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
2019-11-21 11:12:24,684 [http-nio-8045-exec-3] ERROR [,,] o.a.catalina.connector.CoyoteAdapter - Exception while processing an asynchronous request
java.lang.IllegalStateException: Calling [asyncError()] is not valid for a request with Async state [MUST_DISPATCH]
	at org.apache.coyote.AsyncStateMachine.asyncError(AsyncStateMachine.java:440)
	at org.apache.coyote.AbstractProcessor.action(AbstractProcessor.java:512)
	at org.apache.coyote.Request.action(Request.java:430)
	at org.apache.catalina.core.AsyncContextImpl.setErrorState(AsyncContextImpl.java:401)
	at org.apache.catalina.connector.CoyoteAdapter.asyncDispatch(CoyoteAdapter.java:239)
	at org.apache.coyote.AbstractProcessor.dispatch(AbstractProcessor.java:241)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:53)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:836)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1747)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
2019-11-21 11:12:25,265 [http-nio-8045-exec-2] ERROR [,,] o.a.catalina.connector.CoyoteAdapter - Exception while processing an asynchronous request
java.lang.IllegalStateException: Calling [asyncError()] is not valid for a request with Async state [MUST_DISPATCH]
	at org.apache.coyote.AsyncStateMachine.asyncError(AsyncStateMachine.java:440)
	at org.apache.coyote.AbstractProcessor.action(AbstractProcessor.java:512)
	at org.apache.coyote.Request.action(Request.java:430)
	at org.apache.catalina.core.AsyncContextImpl.setErrorState(AsyncContextImpl.java:401)
	at org.apache.catalina.connector.CoyoteAdapter.asyncDispatch(CoyoteAdapter.java:239)
	at org.apache.coyote.AbstractProcessor.dispatch(AbstractProcessor.java:241)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:53)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:836)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1747)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
2019-11-21 11:12:25,449 [http-nio-8045-exec-6] ERROR [,,] o.a.catalina.connector.CoyoteAdapter - Exception while processing an asynchronous request
java.lang.IllegalStateException: Calling [asyncError()] is not valid for a request with Async state [MUST_DISPATCH]
	at org.apache.coyote.AsyncStateMachine.asyncError(AsyncStateMachine.java:440)
	at org.apache.coyote.AbstractProcessor.action(AbstractProcessor.java:512)
	at org.apache.coyote.Request.action(Request.java:430)
	at org.apache.catalina.core.AsyncContextImpl.setErrorState(AsyncContextImpl.java:401)
	at org.apache.catalina.connector.CoyoteAdapter.asyncDispatch(CoyoteAdapter.java:239)
	at org.apache.coyote.AbstractProcessor.dispatch(AbstractProcessor.java:241)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:53)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:836)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1747)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
2019-11-21 11:12:25,614 [http-nio-8045-exec-5] ERROR [,,] o.a.catalina.connector.CoyoteAdapter - Exception while processing an asynchronous request
java.lang.IllegalStateException: Calling [asyncError()] is not valid for a request with Async state [MUST_DISPATCH]
	at org.apache.coyote.AsyncStateMachine.asyncError(AsyncStateMachine.java:440)
	at org.apache.coyote.AbstractProcessor.action(AbstractProcessor.java:512)
	at org.apache.coyote.Request.action(Request.java:430)
	at org.apache.catalina.core.AsyncContextImpl.setErrorState(AsyncContextImpl.java:401)
	at org.apache.catalina.connector.CoyoteAdapter.asyncDispatch(CoyoteAdapter.java:239)
	at org.apache.coyote.AbstractProcessor.dispatch(AbstractProcessor.java:241)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:53)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:836)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1747)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
2019-11-21 11:12:25,913 [http-nio-8045-exec-7] ERROR [,,] o.a.catalina.connector.CoyoteAdapter - Exception while processing an asynchronous request
java.lang.IllegalStateException: Calling [asyncError()] is not valid for a request with Async state [MUST_DISPATCH]
	at org.apache.coyote.AsyncStateMachine.asyncError(AsyncStateMachine.java:440)
	at org.apache.coyote.AbstractProcessor.action(AbstractProcessor.java:512)
	at org.apache.coyote.Request.action(Request.java:430)
	at org.apache.catalina.core.AsyncContextImpl.setErrorState(AsyncContextImpl.java:401)
	at org.apache.catalina.connector.CoyoteAdapter.asyncDispatch(CoyoteAdapter.java:239)
	at org.apache.coyote.AbstractProcessor.dispatch(AbstractProcessor.java:241)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:53)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:836)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1747)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
2019-11-21 11:12:26,101 [http-nio-8045-exec-9] ERROR [,,] o.a.catalina.connector.CoyoteAdapter - Exception while processing an asynchronous request
java.lang.IllegalStateException: Calling [asyncError()] is not valid for a request with Async state [MUST_DISPATCH]
	at org.apache.coyote.AsyncStateMachine.asyncError(AsyncStateMachine.java:440)
	at org.apache.coyote.AbstractProcessor.action(AbstractProcessor.java:512)
	at org.apache.coyote.Request.action(Request.java:430)
	at org.apache.catalina.core.AsyncContextImpl.setErrorState(AsyncContextImpl.java:401)
	at org.apache.catalina.connector.CoyoteAdapter.asyncDispatch(CoyoteAdapter.java:239)
	at org.apache.coyote.AbstractProcessor.dispatch(AbstractProcessor.java:241)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:53)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:836)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1747)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
2019-11-21 11:12:29,487 [http-nio-8045-exec-10] ERROR [,,] o.a.catalina.connector.CoyoteAdapter - Exception while processing an asynchronous request
java.lang.IllegalStateException: Calling [asyncError()] is not valid for a request with Async state [MUST_DISPATCH]
	at org.apache.coyote.AsyncStateMachine.asyncError(AsyncStateMachine.java:440)
	at org.apache.coyote.AbstractProcessor.action(AbstractProcessor.java:512)
	at org.apache.coyote.Request.action(Request.java:430)
	at org.apache.catalina.core.AsyncContextImpl.setErrorState(AsyncContextImpl.java:401)
	at org.apache.catalina.connector.CoyoteAdapter.asyncDispatch(CoyoteAdapter.java:239)
	at org.apache.coyote.AbstractProcessor.dispatch(AbstractProcessor.java:241)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:53)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:836)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1747)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
2019-11-21 11:12:29,678 [http-nio-8045-exec-8] ERROR [,,] o.a.catalina.connector.CoyoteAdapter - Exception while processing an asynchronous request
java.lang.IllegalStateException: Calling [asyncError()] is not valid for a request with Async state [MUST_DISPATCH]
	at org.apache.coyote.AsyncStateMachine.asyncError(AsyncStateMachine.java:440)
	at org.apache.coyote.AbstractProcessor.action(AbstractProcessor.java:512)
	at org.apache.coyote.Request.action(Request.java:430)
	at org.apache.catalina.core.AsyncContextImpl.setErrorState(AsyncContextImpl.java:401)
	at org.apache.catalina.connector.CoyoteAdapter.asyncDispatch(CoyoteAdapter.java:239)
	at org.apache.coyote.AbstractProcessor.dispatch(AbstractProcessor.java:241)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:53)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:836)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1747)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
2019-11-21 11:12:29,839 [http-nio-8045-exec-1] ERROR [,,] o.a.catalina.connector.CoyoteAdapter - Exception while processing an asynchronous request
java.lang.IllegalStateException: Calling [asyncError()] is not valid for a request with Async state [MUST_DISPATCH]
	at org.apache.coyote.AsyncStateMachine.asyncError(AsyncStateMachine.java:440)
	at org.apache.coyote.AbstractProcessor.action(AbstractProcessor.java:512)
	at org.apache.coyote.Request.action(Request.java:430)
	at org.apache.catalina.core.AsyncContextImpl.setErrorState(AsyncContextImpl.java:401)
	at org.apache.catalina.connector.CoyoteAdapter.asyncDispatch(CoyoteAdapter.java:239)
	at org.apache.coyote.AbstractProcessor.dispatch(AbstractProcessor.java:241)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:53)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:836)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1747)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
2019-11-21 11:14:10,002 [http-nio-8045-exec-4] ERROR [,,] o.a.catalina.connector.CoyoteAdapter - Exception while processing an asynchronous request
java.lang.IllegalStateException: Calling [asyncError()] is not valid for a request with Async state [MUST_DISPATCH]
	at org.apache.coyote.AsyncStateMachine.asyncError(AsyncStateMachine.java:440)
	at org.apache.coyote.AbstractProcessor.action(AbstractProcessor.java:512)
	at org.apache.coyote.Request.action(Request.java:430)
	at org.apache.catalina.core.AsyncContextImpl.setErrorState(AsyncContextImpl.java:401)
	at org.apache.catalina.connector.CoyoteAdapter.asyncDispatch(CoyoteAdapter.java:239)
	at org.apache.coyote.AbstractProcessor.dispatch(AbstractProcessor.java:241)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:53)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:836)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1747)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
2019-11-21 11:21:05,664 [http-nio-8045-exec-3] ERROR [,,] o.a.catalina.connector.CoyoteAdapter - Exception while processing an asynchronous request
java.lang.IllegalStateException: Calling [asyncError()] is not valid for a request with Async state [MUST_DISPATCH]
	at org.apache.coyote.AsyncStateMachine.asyncError(AsyncStateMachine.java:440)
	at org.apache.coyote.AbstractProcessor.action(AbstractProcessor.java:512)
	at org.apache.coyote.Request.action(Request.java:430)
	at org.apache.catalina.core.AsyncContextImpl.setErrorState(AsyncContextImpl.java:401)
	at org.apache.catalina.connector.CoyoteAdapter.asyncDispatch(CoyoteAdapter.java:239)
	at org.apache.coyote.AbstractProcessor.dispatch(AbstractProcessor.java:241)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:53)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:836)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1747)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
2019-11-21 11:21:07,754 [http-nio-8045-exec-2] ERROR [,,] o.a.catalina.connector.CoyoteAdapter - Exception while processing an asynchronous request
java.lang.IllegalStateException: Calling [asyncError()] is not valid for a request with Async state [MUST_DISPATCH]
	at org.apache.coyote.AsyncStateMachine.asyncError(AsyncStateMachine.java:440)
	at org.apache.coyote.AbstractProcessor.action(AbstractProcessor.java:512)
	at org.apache.coyote.Request.action(Request.java:430)
	at org.apache.catalina.core.AsyncContextImpl.setErrorState(AsyncContextImpl.java:401)
	at org.apache.catalina.connector.CoyoteAdapter.asyncDispatch(CoyoteAdapter.java:239)
	at org.apache.coyote.AbstractProcessor.dispatch(AbstractProcessor.java:241)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:53)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:836)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1747)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
2019-11-21 11:21:08,069 [http-nio-8045-exec-6] ERROR [,,] o.a.catalina.connector.CoyoteAdapter - Exception while processing an asynchronous request
java.lang.IllegalStateException: Calling [asyncError()] is not valid for a request with Async state [MUST_DISPATCH]
	at org.apache.coyote.AsyncStateMachine.asyncError(AsyncStateMachine.java:440)
	at org.apache.coyote.AbstractProcessor.action(AbstractProcessor.java:512)
	at org.apache.coyote.Request.action(Request.java:430)
	at org.apache.catalina.core.AsyncContextImpl.setErrorState(AsyncContextImpl.java:401)
	at org.apache.catalina.connector.CoyoteAdapter.asyncDispatch(CoyoteAdapter.java:239)
	at org.apache.coyote.AbstractProcessor.dispatch(AbstractProcessor.java:241)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:53)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:836)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1747)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
2019-11-21 11:21:08,234 [http-nio-8045-exec-5] ERROR [,,] o.a.catalina.connector.CoyoteAdapter - Exception while processing an asynchronous request
java.lang.IllegalStateException: Calling [asyncError()] is not valid for a request with Async state [MUST_DISPATCH]
	at org.apache.coyote.AsyncStateMachine.asyncError(AsyncStateMachine.java:440)
	at org.apache.coyote.AbstractProcessor.action(AbstractProcessor.java:512)
	at org.apache.coyote.Request.action(Request.java:430)
	at org.apache.catalina.core.AsyncContextImpl.setErrorState(AsyncContextImpl.java:401)
	at org.apache.catalina.connector.CoyoteAdapter.asyncDispatch(CoyoteAdapter.java:239)
	at org.apache.coyote.AbstractProcessor.dispatch(AbstractProcessor.java:241)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:53)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:836)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1747)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
2019-11-21 11:21:08,399 [http-nio-8045-exec-7] ERROR [,,] o.a.catalina.connector.CoyoteAdapter - Exception while processing an asynchronous request
java.lang.IllegalStateException: Calling [asyncError()] is not valid for a request with Async state [MUST_DISPATCH]
	at org.apache.coyote.AsyncStateMachine.asyncError(AsyncStateMachine.java:440)
	at org.apache.coyote.AbstractProcessor.action(AbstractProcessor.java:512)
	at org.apache.coyote.Request.action(Request.java:430)
	at org.apache.catalina.core.AsyncContextImpl.setErrorState(AsyncContextImpl.java:401)
	at org.apache.catalina.connector.CoyoteAdapter.asyncDispatch(CoyoteAdapter.java:239)
	at org.apache.coyote.AbstractProcessor.dispatch(AbstractProcessor.java:241)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:53)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:836)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1747)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
2019-11-21 11:21:08,547 [http-nio-8045-exec-9] ERROR [,,] o.a.catalina.connector.CoyoteAdapter - Exception while processing an asynchronous request
java.lang.IllegalStateException: Calling [asyncError()] is not valid for a request with Async state [MUST_DISPATCH]
	at org.apache.coyote.AsyncStateMachine.asyncError(AsyncStateMachine.java:440)
	at org.apache.coyote.AbstractProcessor.action(AbstractProcessor.java:512)
	at org.apache.coyote.Request.action(Request.java:430)
	at org.apache.catalina.core.AsyncContextImpl.setErrorState(AsyncContextImpl.java:401)
	at org.apache.catalina.connector.CoyoteAdapter.asyncDispatch(CoyoteAdapter.java:239)
	at org.apache.coyote.AbstractProcessor.dispatch(AbstractProcessor.java:241)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:53)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:836)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1747)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
2019-11-21 11:21:08,699 [http-nio-8045-exec-10] ERROR [,,] o.a.catalina.connector.CoyoteAdapter - Exception while processing an asynchronous request
java.lang.IllegalStateException: Calling [asyncError()] is not valid for a request with Async state [MUST_DISPATCH]
	at org.apache.coyote.AsyncStateMachine.asyncError(AsyncStateMachine.java:440)
	at org.apache.coyote.AbstractProcessor.action(AbstractProcessor.java:512)
	at org.apache.coyote.Request.action(Request.java:430)
	at org.apache.catalina.core.AsyncContextImpl.setErrorState(AsyncContextImpl.java:401)
	at org.apache.catalina.connector.CoyoteAdapter.asyncDispatch(CoyoteAdapter.java:239)
	at org.apache.coyote.AbstractProcessor.dispatch(AbstractProcessor.java:241)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:53)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:836)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1747)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
2019-11-21 11:21:17,990 [Thread-79] INFO  [,,] o.s.s.c.ThreadPoolTaskExecutor - Shutting down ExecutorService 'applicationTaskExecutor'
2019-11-21 11:21:17,996 [Thread-79] INFO  [,,] o.s.o.j.LocalContainerEntityManagerFactoryBean - Closing JPA EntityManagerFactory for persistence unit 'default'
2019-11-21 11:21:18,010 [Thread-79] INFO  [,,] com.zaxxer.hikari.HikariDataSource - HikariPool-1 - Shutdown initiated...
2019-11-21 11:21:18,063 [Thread-79] INFO  [,,] com.zaxxer.hikari.HikariDataSource - HikariPool-1 - Shutdown completed.
2019-11-21 11:21:52,323 [restartedMain] INFO  [,,] com.xiao.emsoft.DemoApplication - Starting DemoApplication on xiaohanfengdeMacBook-Pro.local with PID 13163 (started by xiaohanfeng in /Users/xiaohanfeng/Documents/代码库/账户支付系统/testing)
2019-11-21 11:21:52,326 [restartedMain] INFO  [,,] com.xiao.emsoft.DemoApplication - The following profiles are active: stream,database,cloud
2019-11-21 11:21:52,364 [restartedMain] INFO  [,,] o.s.b.d.e.DevToolsPropertyDefaultsPostProcessor - Devtools property defaults active! Set 'spring.devtools.add-properties' to 'false' to disable
2019-11-21 11:21:52,364 [restartedMain] INFO  [,,] o.s.b.d.e.DevToolsPropertyDefaultsPostProcessor - For additional web related logging consider setting the 'logging.level.web' property to 'DEBUG'
2019-11-21 11:21:53,239 [restartedMain] INFO  [,,] o.s.d.r.c.RepositoryConfigurationDelegate - Bootstrapping Spring Data repositories in DEFAULT mode.
2019-11-21 11:21:53,316 [restartedMain] INFO  [,,] o.s.d.r.c.RepositoryConfigurationDelegate - Finished Spring Data repository scanning in 73ms. Found 1 repository interfaces.
2019-11-21 11:21:53,632 [restartedMain] INFO  [,,] o.s.c.s.PostProcessorRegistrationDelegate$BeanPostProcessorChecker - Bean 'org.springframework.transaction.annotation.ProxyTransactionManagementConfiguration' of type [org.springframework.transaction.annotation.ProxyTransactionManagementConfiguration$$EnhancerBySpringCGLIB$$179e2c18] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2019-11-21 11:21:54,037 [restartedMain] INFO  [,,] o.s.b.w.e.tomcat.TomcatWebServer - Tomcat initialized with port(s): 8045 (http)
2019-11-21 11:21:54,047 [restartedMain] INFO  [,,] o.a.coyote.http11.Http11NioProtocol - Initializing ProtocolHandler ["http-nio-8045"]
2019-11-21 11:21:54,054 [restartedMain] INFO  [,,] o.a.catalina.core.StandardService - Starting service [Tomcat]
2019-11-21 11:21:54,054 [restartedMain] INFO  [,,] o.a.catalina.core.StandardEngine - Starting Servlet engine: [Apache Tomcat/9.0.19]
2019-11-21 11:21:54,142 [restartedMain] INFO  [,,] o.a.c.c.C.[Tomcat].[localhost].[/] - Initializing Spring embedded WebApplicationContext
2019-11-21 11:21:54,142 [restartedMain] INFO  [,,] o.s.web.context.ContextLoader - Root WebApplicationContext: initialization completed in 1778 ms
2019-11-21 11:21:54,576 [restartedMain] INFO  [,,] com.zaxxer.hikari.HikariDataSource - HikariPool-1 - Starting...
2019-11-21 11:21:59,851 [restartedMain] INFO  [,,] com.zaxxer.hikari.HikariDataSource - HikariPool-1 - Start completed.
2019-11-21 11:21:59,891 [restartedMain] INFO  [,,] o.h.jpa.internal.util.LogHelper - HHH000204: Processing PersistenceUnitInfo [name: default]
2019-11-21 11:21:59,940 [restartedMain] INFO  [,,] org.hibernate.Version - HHH000412: Hibernate Core {5.4.5.Final}
2019-11-21 11:22:00,039 [restartedMain] INFO  [,,] o.h.annotations.common.Version - HCANN000001: Hibernate Commons Annotations {5.1.0.Final}
2019-11-21 11:22:00,142 [restartedMain] INFO  [,,] org.hibernate.dialect.Dialect - HHH000400: Using dialect: org.hibernate.dialect.Oracle10gDialect
2019-11-21 11:22:03,592 [restartedMain] INFO  [,,] o.h.e.t.j.p.i.JtaPlatformInitiator - HHH000490: Using JtaPlatform implementation: [org.hibernate.engine.transaction.jta.platform.internal.NoJtaPlatform]
2019-11-21 11:22:03,599 [restartedMain] INFO  [,,] o.s.o.j.LocalContainerEntityManagerFactoryBean - Initialized JPA EntityManagerFactory for persistence unit 'default'
2019-11-21 11:22:03,641 [restartedMain] INFO  [,,] o.s.b.d.a.OptionalLiveReloadServer - LiveReload server is running on port 35729
2019-11-21 11:22:04,167 [restartedMain] WARN  [,,] o.s.b.a.o.j.JpaBaseConfiguration$JpaWebConfiguration$JpaWebMvcConfiguration - spring.jpa.open-in-view is enabled by default. Therefore, database queries may be performed during view rendering. Explicitly configure spring.jpa.open-in-view to disable this warning
2019-11-21 11:22:04,312 [restartedMain] INFO  [,,] o.s.b.a.e.web.EndpointLinksResolver - Exposing 2 endpoint(s) beneath base path '/actuator'
2019-11-21 11:22:04,383 [restartedMain] INFO  [,,] s.d.s.w.PropertySourcedRequestMappingHandlerMapping - Mapped URL path [/v2/api-docs] onto method [public org.springframework.http.ResponseEntity<springfox.documentation.spring.web.json.Json> springfox.documentation.swagger2.web.Swagger2Controller.getDocumentation(java.lang.String,javax.servlet.http.HttpServletRequest)]
2019-11-21 11:22:04,548 [restartedMain] INFO  [,,] o.s.s.c.ThreadPoolTaskExecutor - Initializing ExecutorService 'applicationTaskExecutor'
2019-11-21 11:22:04,641 [restartedMain] WARN  [,,] o.s.b.a.t.ThymeleafAutoConfiguration$DefaultTemplateResolverConfiguration - Cannot find template location: classpath:/templates/ (please add some templates or check your Thymeleaf configuration)
2019-11-21 11:22:05,014 [restartedMain] WARN  [,,] o.s.b.a.f.FreeMarkerAutoConfiguration - Cannot find template location(s): [classpath:/templates/] (please add some templates, check your FreeMarker configuration, or set spring.freemarker.checkTemplateLocation=false)
2019-11-21 11:22:05,117 [restartedMain] INFO  [,,] s.d.s.w.p.DocumentationPluginsBootstrapper - Context refreshed
2019-11-21 11:22:05,132 [restartedMain] INFO  [,,] s.d.s.w.p.DocumentationPluginsBootstrapper - Found 1 custom documentation plugin(s)
2019-11-21 11:22:05,166 [restartedMain] INFO  [,,] s.d.s.w.s.ApiListingReferenceScanner - Scanning for api listing references
2019-11-21 11:22:05,242 [restartedMain] INFO  [,,] o.a.coyote.http11.Http11NioProtocol - Starting ProtocolHandler ["http-nio-8045"]
2019-11-21 11:22:05,260 [restartedMain] INFO  [,,] o.s.b.w.e.tomcat.TomcatWebServer - Tomcat started on port(s): 8045 (http) with context path ''
2019-11-21 11:22:05,263 [restartedMain] INFO  [,,] com.xiao.emsoft.DemoApplication - Started DemoApplication in 33.336 seconds (JVM running for 39.07)
2019-11-21 11:22:05,391 [RMI TCP Connection(3)-192.168.31.108] INFO  [,,] o.a.c.c.C.[Tomcat].[localhost].[/] - Initializing Spring DispatcherServlet 'dispatcherServlet'
2019-11-21 11:22:05,392 [RMI TCP Connection(3)-192.168.31.108] INFO  [,,] o.s.web.servlet.DispatcherServlet - Initializing Servlet 'dispatcherServlet'
2019-11-21 11:22:05,402 [RMI TCP Connection(3)-192.168.31.108] INFO  [,,] o.s.web.servlet.DispatcherServlet - Completed initialization in 10 ms
2019-11-21 11:22:37,063 [http-nio-8045-exec-3] ERROR [,,] o.a.catalina.connector.CoyoteAdapter - Exception while processing an asynchronous request
java.lang.IllegalStateException: Calling [asyncError()] is not valid for a request with Async state [MUST_DISPATCH]
	at org.apache.coyote.AsyncStateMachine.asyncError(AsyncStateMachine.java:440)
	at org.apache.coyote.AbstractProcessor.action(AbstractProcessor.java:512)
	at org.apache.coyote.Request.action(Request.java:430)
	at org.apache.catalina.core.AsyncContextImpl.setErrorState(AsyncContextImpl.java:401)
	at org.apache.catalina.connector.CoyoteAdapter.asyncDispatch(CoyoteAdapter.java:239)
	at org.apache.coyote.AbstractProcessor.dispatch(AbstractProcessor.java:241)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:53)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:836)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1747)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
2019-11-21 11:22:44,713 [http-nio-8045-exec-4] ERROR [,,] o.a.catalina.connector.CoyoteAdapter - Exception while processing an asynchronous request
java.lang.IllegalStateException: Calling [asyncError()] is not valid for a request with Async state [MUST_DISPATCH]
	at org.apache.coyote.AsyncStateMachine.asyncError(AsyncStateMachine.java:440)
	at org.apache.coyote.AbstractProcessor.action(AbstractProcessor.java:512)
	at org.apache.coyote.Request.action(Request.java:430)
	at org.apache.catalina.core.AsyncContextImpl.setErrorState(AsyncContextImpl.java:401)
	at org.apache.catalina.connector.CoyoteAdapter.asyncDispatch(CoyoteAdapter.java:239)
	at org.apache.coyote.AbstractProcessor.dispatch(AbstractProcessor.java:241)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:53)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:836)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1747)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
2019-11-21 11:22:45,333 [http-nio-8045-exec-5] ERROR [,,] o.a.catalina.connector.CoyoteAdapter - Exception while processing an asynchronous request
java.lang.IllegalStateException: Calling [asyncError()] is not valid for a request with Async state [MUST_DISPATCH]
	at org.apache.coyote.AsyncStateMachine.asyncError(AsyncStateMachine.java:440)
	at org.apache.coyote.AbstractProcessor.action(AbstractProcessor.java:512)
	at org.apache.coyote.Request.action(Request.java:430)
	at org.apache.catalina.core.AsyncContextImpl.setErrorState(AsyncContextImpl.java:401)
	at org.apache.catalina.connector.CoyoteAdapter.asyncDispatch(CoyoteAdapter.java:239)
	at org.apache.coyote.AbstractProcessor.dispatch(AbstractProcessor.java:241)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:53)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:836)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1747)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
2019-11-21 11:22:45,503 [http-nio-8045-exec-7] ERROR [,,] o.a.catalina.connector.CoyoteAdapter - Exception while processing an asynchronous request
java.lang.IllegalStateException: Calling [asyncError()] is not valid for a request with Async state [MUST_DISPATCH]
	at org.apache.coyote.AsyncStateMachine.asyncError(AsyncStateMachine.java:440)
	at org.apache.coyote.AbstractProcessor.action(AbstractProcessor.java:512)
	at org.apache.coyote.Request.action(Request.java:430)
	at org.apache.catalina.core.AsyncContextImpl.setErrorState(AsyncContextImpl.java:401)
	at org.apache.catalina.connector.CoyoteAdapter.asyncDispatch(CoyoteAdapter.java:239)
	at org.apache.coyote.AbstractProcessor.dispatch(AbstractProcessor.java:241)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:53)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:836)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1747)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
2019-11-21 11:22:48,436 [http-nio-8045-exec-6] ERROR [,,] o.a.catalina.connector.CoyoteAdapter - Exception while processing an asynchronous request
java.lang.IllegalStateException: Calling [asyncError()] is not valid for a request with Async state [MUST_DISPATCH]
	at org.apache.coyote.AsyncStateMachine.asyncError(AsyncStateMachine.java:440)
	at org.apache.coyote.AbstractProcessor.action(AbstractProcessor.java:512)
	at org.apache.coyote.Request.action(Request.java:430)
	at org.apache.catalina.core.AsyncContextImpl.setErrorState(AsyncContextImpl.java:401)
	at org.apache.catalina.connector.CoyoteAdapter.asyncDispatch(CoyoteAdapter.java:239)
	at org.apache.coyote.AbstractProcessor.dispatch(AbstractProcessor.java:241)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:53)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:836)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1747)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
2019-11-21 11:22:48,750 [http-nio-8045-exec-8] ERROR [,,] o.a.catalina.connector.CoyoteAdapter - Exception while processing an asynchronous request
java.lang.IllegalStateException: Calling [asyncError()] is not valid for a request with Async state [MUST_DISPATCH]
	at org.apache.coyote.AsyncStateMachine.asyncError(AsyncStateMachine.java:440)
	at org.apache.coyote.AbstractProcessor.action(AbstractProcessor.java:512)
	at org.apache.coyote.Request.action(Request.java:430)
	at org.apache.catalina.core.AsyncContextImpl.setErrorState(AsyncContextImpl.java:401)
	at org.apache.catalina.connector.CoyoteAdapter.asyncDispatch(CoyoteAdapter.java:239)
	at org.apache.coyote.AbstractProcessor.dispatch(AbstractProcessor.java:241)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:53)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:836)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1747)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
2019-11-21 11:22:48,955 [http-nio-8045-exec-9] ERROR [,,] o.a.catalina.connector.CoyoteAdapter - Exception while processing an asynchronous request
java.lang.IllegalStateException: Calling [asyncError()] is not valid for a request with Async state [MUST_DISPATCH]
	at org.apache.coyote.AsyncStateMachine.asyncError(AsyncStateMachine.java:440)
	at org.apache.coyote.AbstractProcessor.action(AbstractProcessor.java:512)
	at org.apache.coyote.Request.action(Request.java:430)
	at org.apache.catalina.core.AsyncContextImpl.setErrorState(AsyncContextImpl.java:401)
	at org.apache.catalina.connector.CoyoteAdapter.asyncDispatch(CoyoteAdapter.java:239)
	at org.apache.coyote.AbstractProcessor.dispatch(AbstractProcessor.java:241)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:53)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:836)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1747)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
2019-11-21 11:22:49,128 [http-nio-8045-exec-10] ERROR [,,] o.a.catalina.connector.CoyoteAdapter - Exception while processing an asynchronous request
java.lang.IllegalStateException: Calling [asyncError()] is not valid for a request with Async state [MUST_DISPATCH]
	at org.apache.coyote.AsyncStateMachine.asyncError(AsyncStateMachine.java:440)
	at org.apache.coyote.AbstractProcessor.action(AbstractProcessor.java:512)
	at org.apache.coyote.Request.action(Request.java:430)
	at org.apache.catalina.core.AsyncContextImpl.setErrorState(AsyncContextImpl.java:401)
	at org.apache.catalina.connector.CoyoteAdapter.asyncDispatch(CoyoteAdapter.java:239)
	at org.apache.coyote.AbstractProcessor.dispatch(AbstractProcessor.java:241)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:53)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:836)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1747)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
2019-11-21 11:22:49,319 [http-nio-8045-exec-1] ERROR [,,] o.a.catalina.connector.CoyoteAdapter - Exception while processing an asynchronous request
java.lang.IllegalStateException: Calling [asyncError()] is not valid for a request with Async state [MUST_DISPATCH]
	at org.apache.coyote.AsyncStateMachine.asyncError(AsyncStateMachine.java:440)
	at org.apache.coyote.AbstractProcessor.action(AbstractProcessor.java:512)
	at org.apache.coyote.Request.action(Request.java:430)
	at org.apache.catalina.core.AsyncContextImpl.setErrorState(AsyncContextImpl.java:401)
	at org.apache.catalina.connector.CoyoteAdapter.asyncDispatch(CoyoteAdapter.java:239)
	at org.apache.coyote.AbstractProcessor.dispatch(AbstractProcessor.java:241)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:53)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:836)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1747)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
2019-11-21 11:22:49,515 [http-nio-8045-exec-2] ERROR [,,] o.a.catalina.connector.CoyoteAdapter - Exception while processing an asynchronous request
java.lang.IllegalStateException: Calling [asyncError()] is not valid for a request with Async state [MUST_DISPATCH]
	at org.apache.coyote.AsyncStateMachine.asyncError(AsyncStateMachine.java:440)
	at org.apache.coyote.AbstractProcessor.action(AbstractProcessor.java:512)
	at org.apache.coyote.Request.action(Request.java:430)
	at org.apache.catalina.core.AsyncContextImpl.setErrorState(AsyncContextImpl.java:401)
	at org.apache.catalina.connector.CoyoteAdapter.asyncDispatch(CoyoteAdapter.java:239)
	at org.apache.coyote.AbstractProcessor.dispatch(AbstractProcessor.java:241)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:53)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:836)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1747)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
2019-11-21 11:26:09,721 [http-nio-8045-exec-3] ERROR [,,] o.a.catalina.connector.CoyoteAdapter - Exception while processing an asynchronous request
java.lang.IllegalStateException: Calling [asyncError()] is not valid for a request with Async state [MUST_DISPATCH]
	at org.apache.coyote.AsyncStateMachine.asyncError(AsyncStateMachine.java:440)
	at org.apache.coyote.AbstractProcessor.action(AbstractProcessor.java:512)
	at org.apache.coyote.Request.action(Request.java:430)
	at org.apache.catalina.core.AsyncContextImpl.setErrorState(AsyncContextImpl.java:401)
	at org.apache.catalina.connector.CoyoteAdapter.asyncDispatch(CoyoteAdapter.java:239)
	at org.apache.coyote.AbstractProcessor.dispatch(AbstractProcessor.java:241)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:53)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:836)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1747)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
2019-11-21 11:26:15,365 [http-nio-8045-exec-4] ERROR [,,] o.a.catalina.connector.CoyoteAdapter - Exception while processing an asynchronous request
java.lang.IllegalStateException: Calling [asyncError()] is not valid for a request with Async state [MUST_DISPATCH]
	at org.apache.coyote.AsyncStateMachine.asyncError(AsyncStateMachine.java:440)
	at org.apache.coyote.AbstractProcessor.action(AbstractProcessor.java:512)
	at org.apache.coyote.Request.action(Request.java:430)
	at org.apache.catalina.core.AsyncContextImpl.setErrorState(AsyncContextImpl.java:401)
	at org.apache.catalina.connector.CoyoteAdapter.asyncDispatch(CoyoteAdapter.java:239)
	at org.apache.coyote.AbstractProcessor.dispatch(AbstractProcessor.java:241)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:53)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:836)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1747)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
2019-11-21 11:26:16,178 [http-nio-8045-exec-5] ERROR [,,] o.a.catalina.connector.CoyoteAdapter - Exception while processing an asynchronous request
java.lang.IllegalStateException: Calling [asyncError()] is not valid for a request with Async state [MUST_DISPATCH]
	at org.apache.coyote.AsyncStateMachine.asyncError(AsyncStateMachine.java:440)
	at org.apache.coyote.AbstractProcessor.action(AbstractProcessor.java:512)
	at org.apache.coyote.Request.action(Request.java:430)
	at org.apache.catalina.core.AsyncContextImpl.setErrorState(AsyncContextImpl.java:401)
	at org.apache.catalina.connector.CoyoteAdapter.asyncDispatch(CoyoteAdapter.java:239)
	at org.apache.coyote.AbstractProcessor.dispatch(AbstractProcessor.java:241)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:53)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:836)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1747)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
2019-11-21 11:26:16,421 [http-nio-8045-exec-7] ERROR [,,] o.a.catalina.connector.CoyoteAdapter - Exception while processing an asynchronous request
java.lang.IllegalStateException: Calling [asyncError()] is not valid for a request with Async state [MUST_DISPATCH]
	at org.apache.coyote.AsyncStateMachine.asyncError(AsyncStateMachine.java:440)
	at org.apache.coyote.AbstractProcessor.action(AbstractProcessor.java:512)
	at org.apache.coyote.Request.action(Request.java:430)
	at org.apache.catalina.core.AsyncContextImpl.setErrorState(AsyncContextImpl.java:401)
	at org.apache.catalina.connector.CoyoteAdapter.asyncDispatch(CoyoteAdapter.java:239)
	at org.apache.coyote.AbstractProcessor.dispatch(AbstractProcessor.java:241)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:53)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:836)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1747)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
2019-11-21 11:26:16,592 [http-nio-8045-exec-6] ERROR [,,] o.a.catalina.connector.CoyoteAdapter - Exception while processing an asynchronous request
java.lang.IllegalStateException: Calling [asyncError()] is not valid for a request with Async state [MUST_DISPATCH]
	at org.apache.coyote.AsyncStateMachine.asyncError(AsyncStateMachine.java:440)
	at org.apache.coyote.AbstractProcessor.action(AbstractProcessor.java:512)
	at org.apache.coyote.Request.action(Request.java:430)
	at org.apache.catalina.core.AsyncContextImpl.setErrorState(AsyncContextImpl.java:401)
	at org.apache.catalina.connector.CoyoteAdapter.asyncDispatch(CoyoteAdapter.java:239)
	at org.apache.coyote.AbstractProcessor.dispatch(AbstractProcessor.java:241)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:53)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:836)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1747)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
2019-11-21 11:26:16,796 [http-nio-8045-exec-8] ERROR [,,] o.a.catalina.connector.CoyoteAdapter - Exception while processing an asynchronous request
java.lang.IllegalStateException: Calling [asyncError()] is not valid for a request with Async state [MUST_DISPATCH]
	at org.apache.coyote.AsyncStateMachine.asyncError(AsyncStateMachine.java:440)
	at org.apache.coyote.AbstractProcessor.action(AbstractProcessor.java:512)
	at org.apache.coyote.Request.action(Request.java:430)
	at org.apache.catalina.core.AsyncContextImpl.setErrorState(AsyncContextImpl.java:401)
	at org.apache.catalina.connector.CoyoteAdapter.asyncDispatch(CoyoteAdapter.java:239)
	at org.apache.coyote.AbstractProcessor.dispatch(AbstractProcessor.java:241)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:53)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:836)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1747)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
2019-11-21 11:38:50,364 [Thread-79] INFO  [,,] o.s.s.c.ThreadPoolTaskExecutor - Shutting down ExecutorService 'applicationTaskExecutor'
2019-11-21 11:38:50,366 [Thread-79] INFO  [,,] o.s.o.j.LocalContainerEntityManagerFactoryBean - Closing JPA EntityManagerFactory for persistence unit 'default'
2019-11-21 11:38:50,370 [Thread-79] INFO  [,,] com.zaxxer.hikari.HikariDataSource - HikariPool-1 - Shutdown initiated...
2019-11-21 11:38:50,430 [Thread-79] INFO  [,,] com.zaxxer.hikari.HikariDataSource - HikariPool-1 - Shutdown completed.
2019-11-21 11:39:34,448 [restartedMain] INFO  [,,] com.xiao.emsoft.DemoApplication - Starting DemoApplication on xiaohanfengdeMacBook-Pro.local with PID 13200 (started by xiaohanfeng in /Users/xiaohanfeng/Documents/代码库/账户支付系统/testing)
2019-11-21 11:39:34,450 [restartedMain] INFO  [,,] com.xiao.emsoft.DemoApplication - The following profiles are active: stream,database,cloud
2019-11-21 11:39:34,484 [restartedMain] INFO  [,,] o.s.b.d.e.DevToolsPropertyDefaultsPostProcessor - Devtools property defaults active! Set 'spring.devtools.add-properties' to 'false' to disable
2019-11-21 11:39:34,485 [restartedMain] INFO  [,,] o.s.b.d.e.DevToolsPropertyDefaultsPostProcessor - For additional web related logging consider setting the 'logging.level.web' property to 'DEBUG'
2019-11-21 11:39:35,718 [restartedMain] INFO  [,,] o.s.d.r.c.RepositoryConfigurationDelegate - Bootstrapping Spring Data repositories in DEFAULT mode.
2019-11-21 11:39:35,815 [restartedMain] INFO  [,,] o.s.d.r.c.RepositoryConfigurationDelegate - Finished Spring Data repository scanning in 90ms. Found 1 repository interfaces.
2019-11-21 11:39:36,197 [restartedMain] INFO  [,,] o.s.c.s.PostProcessorRegistrationDelegate$BeanPostProcessorChecker - Bean 'org.springframework.transaction.annotation.ProxyTransactionManagementConfiguration' of type [org.springframework.transaction.annotation.ProxyTransactionManagementConfiguration$$EnhancerBySpringCGLIB$$4f20ebe0] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2019-11-21 11:39:36,710 [restartedMain] INFO  [,,] o.s.b.w.e.tomcat.TomcatWebServer - Tomcat initialized with port(s): 8045 (http)
2019-11-21 11:39:36,721 [restartedMain] INFO  [,,] o.a.coyote.http11.Http11NioProtocol - Initializing ProtocolHandler ["http-nio-8045"]
2019-11-21 11:39:36,727 [restartedMain] INFO  [,,] o.a.catalina.core.StandardService - Starting service [Tomcat]
2019-11-21 11:39:36,728 [restartedMain] INFO  [,,] o.a.catalina.core.StandardEngine - Starting Servlet engine: [Apache Tomcat/9.0.19]
2019-11-21 11:39:36,819 [restartedMain] INFO  [,,] o.a.c.c.C.[Tomcat].[localhost].[/] - Initializing Spring embedded WebApplicationContext
2019-11-21 11:39:36,819 [restartedMain] INFO  [,,] o.s.web.context.ContextLoader - Root WebApplicationContext: initialization completed in 2334 ms
2019-11-21 11:39:37,285 [restartedMain] INFO  [,,] com.zaxxer.hikari.HikariDataSource - HikariPool-1 - Starting...
2019-11-21 11:39:42,574 [restartedMain] INFO  [,,] com.zaxxer.hikari.HikariDataSource - HikariPool-1 - Start completed.
2019-11-21 11:39:42,637 [restartedMain] INFO  [,,] o.h.jpa.internal.util.LogHelper - HHH000204: Processing PersistenceUnitInfo [name: default]
2019-11-21 11:39:42,713 [restartedMain] INFO  [,,] org.hibernate.Version - HHH000412: Hibernate Core {5.4.5.Final}
2019-11-21 11:39:42,885 [restartedMain] INFO  [,,] o.h.annotations.common.Version - HCANN000001: Hibernate Commons Annotations {5.1.0.Final}
2019-11-21 11:39:43,152 [restartedMain] INFO  [,,] org.hibernate.dialect.Dialect - HHH000400: Using dialect: org.hibernate.dialect.Oracle10gDialect
2019-11-21 11:39:47,631 [restartedMain] INFO  [,,] o.h.e.t.j.p.i.JtaPlatformInitiator - HHH000490: Using JtaPlatform implementation: [org.hibernate.engine.transaction.jta.platform.internal.NoJtaPlatform]
2019-11-21 11:39:47,639 [restartedMain] INFO  [,,] o.s.o.j.LocalContainerEntityManagerFactoryBean - Initialized JPA EntityManagerFactory for persistence unit 'default'
2019-11-21 11:39:47,679 [restartedMain] WARN  [,,] o.s.b.d.a.OptionalLiveReloadServer - Unable to start LiveReload server
2019-11-21 11:39:48,216 [restartedMain] WARN  [,,] o.s.b.a.o.j.JpaBaseConfiguration$JpaWebConfiguration$JpaWebMvcConfiguration - spring.jpa.open-in-view is enabled by default. Therefore, database queries may be performed during view rendering. Explicitly configure spring.jpa.open-in-view to disable this warning
2019-11-21 11:39:48,360 [restartedMain] INFO  [,,] o.s.b.a.e.web.EndpointLinksResolver - Exposing 2 endpoint(s) beneath base path '/actuator'
2019-11-21 11:39:48,532 [restartedMain] INFO  [,,] s.d.s.w.PropertySourcedRequestMappingHandlerMapping - Mapped URL path [/v2/api-docs] onto method [public org.springframework.http.ResponseEntity<springfox.documentation.spring.web.json.Json> springfox.documentation.swagger2.web.Swagger2Controller.getDocumentation(java.lang.String,javax.servlet.http.HttpServletRequest)]
2019-11-21 11:39:48,808 [restartedMain] INFO  [,,] o.s.s.c.ThreadPoolTaskExecutor - Initializing ExecutorService 'applicationTaskExecutor'
2019-11-21 11:39:48,892 [restartedMain] WARN  [,,] o.s.b.a.t.ThymeleafAutoConfiguration$DefaultTemplateResolverConfiguration - Cannot find template location: classpath:/templates/ (please add some templates or check your Thymeleaf configuration)
2019-11-21 11:39:49,102 [restartedMain] WARN  [,,] o.s.b.a.f.FreeMarkerAutoConfiguration - Cannot find template location(s): [classpath:/templates/] (please add some templates, check your FreeMarker configuration, or set spring.freemarker.checkTemplateLocation=false)
2019-11-21 11:39:49,281 [restartedMain] INFO  [,,] s.d.s.w.p.DocumentationPluginsBootstrapper - Context refreshed
2019-11-21 11:39:49,301 [restartedMain] INFO  [,,] s.d.s.w.p.DocumentationPluginsBootstrapper - Found 1 custom documentation plugin(s)
2019-11-21 11:39:49,338 [restartedMain] INFO  [,,] s.d.s.w.s.ApiListingReferenceScanner - Scanning for api listing references
2019-11-21 11:39:49,418 [restartedMain] INFO  [,,] o.a.coyote.http11.Http11NioProtocol - Starting ProtocolHandler ["http-nio-8045"]
2019-11-21 11:39:49,437 [restartedMain] INFO  [,,] o.s.b.w.e.tomcat.TomcatWebServer - Tomcat started on port(s): 8045 (http) with context path ''
2019-11-21 11:39:49,440 [restartedMain] INFO  [,,] com.xiao.emsoft.DemoApplication - Started DemoApplication in 35.411 seconds (JVM running for 41.18)
2019-11-21 11:39:49,528 [RMI TCP Connection(4)-192.168.31.108] INFO  [,,] o.a.c.c.C.[Tomcat].[localhost].[/] - Initializing Spring DispatcherServlet 'dispatcherServlet'
2019-11-21 11:39:49,529 [RMI TCP Connection(4)-192.168.31.108] INFO  [,,] o.s.web.servlet.DispatcherServlet - Initializing Servlet 'dispatcherServlet'
2019-11-21 11:39:49,539 [RMI TCP Connection(4)-192.168.31.108] INFO  [,,] o.s.web.servlet.DispatcherServlet - Completed initialization in 10 ms
2019-11-21 11:44:29,475 [Thread-82] INFO  [,,] o.s.s.c.ThreadPoolTaskExecutor - Shutting down ExecutorService 'applicationTaskExecutor'
2019-11-21 11:44:29,479 [Thread-82] INFO  [,,] o.s.o.j.LocalContainerEntityManagerFactoryBean - Closing JPA EntityManagerFactory for persistence unit 'default'
2019-11-21 11:44:29,485 [Thread-82] INFO  [,,] com.zaxxer.hikari.HikariDataSource - HikariPool-1 - Shutdown initiated...
2019-11-21 11:44:29,521 [Thread-82] INFO  [,,] com.zaxxer.hikari.HikariDataSource - HikariPool-1 - Shutdown completed.
2019-11-21 11:45:02,485 [restartedMain] INFO  [,,] com.xiao.emsoft.DemoApplication - Starting DemoApplication on xiaohanfengdeMacBook-Pro.local with PID 13225 (started by xiaohanfeng in /Users/xiaohanfeng/Documents/代码库/账户支付系统/testing)
2019-11-21 11:45:02,487 [restartedMain] INFO  [,,] com.xiao.emsoft.DemoApplication - The following profiles are active: stream,database,cloud
2019-11-21 11:45:02,522 [restartedMain] INFO  [,,] o.s.b.d.e.DevToolsPropertyDefaultsPostProcessor - Devtools property defaults active! Set 'spring.devtools.add-properties' to 'false' to disable
2019-11-21 11:45:02,523 [restartedMain] INFO  [,,] o.s.b.d.e.DevToolsPropertyDefaultsPostProcessor - For additional web related logging consider setting the 'logging.level.web' property to 'DEBUG'
2019-11-21 11:45:03,410 [restartedMain] INFO  [,,] o.s.d.r.c.RepositoryConfigurationDelegate - Bootstrapping Spring Data repositories in DEFAULT mode.
2019-11-21 11:45:03,491 [restartedMain] INFO  [,,] o.s.d.r.c.RepositoryConfigurationDelegate - Finished Spring Data repository scanning in 76ms. Found 1 repository interfaces.
2019-11-21 11:45:03,806 [restartedMain] INFO  [,,] o.s.c.s.PostProcessorRegistrationDelegate$BeanPostProcessorChecker - Bean 'org.springframework.transaction.annotation.ProxyTransactionManagementConfiguration' of type [org.springframework.transaction.annotation.ProxyTransactionManagementConfiguration$$EnhancerBySpringCGLIB$$7532f902] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2019-11-21 11:45:04,202 [restartedMain] INFO  [,,] o.s.b.w.e.tomcat.TomcatWebServer - Tomcat initialized with port(s): 8045 (http)
2019-11-21 11:45:04,212 [restartedMain] INFO  [,,] o.a.coyote.http11.Http11NioProtocol - Initializing ProtocolHandler ["http-nio-8045"]
2019-11-21 11:45:04,217 [restartedMain] INFO  [,,] o.a.catalina.core.StandardService - Starting service [Tomcat]
2019-11-21 11:45:04,218 [restartedMain] INFO  [,,] o.a.catalina.core.StandardEngine - Starting Servlet engine: [Apache Tomcat/9.0.19]
2019-11-21 11:45:04,300 [restartedMain] INFO  [,,] o.a.c.c.C.[Tomcat].[localhost].[/] - Initializing Spring embedded WebApplicationContext
2019-11-21 11:45:04,300 [restartedMain] INFO  [,,] o.s.web.context.ContextLoader - Root WebApplicationContext: initialization completed in 1777 ms
2019-11-21 11:45:04,728 [restartedMain] INFO  [,,] com.zaxxer.hikari.HikariDataSource - HikariPool-1 - Starting...
2019-11-21 11:45:09,982 [restartedMain] INFO  [,,] com.zaxxer.hikari.HikariDataSource - HikariPool-1 - Start completed.
2019-11-21 11:45:10,020 [restartedMain] INFO  [,,] o.h.jpa.internal.util.LogHelper - HHH000204: Processing PersistenceUnitInfo [name: default]
2019-11-21 11:45:10,064 [restartedMain] INFO  [,,] org.hibernate.Version - HHH000412: Hibernate Core {5.4.5.Final}
2019-11-21 11:45:10,147 [restartedMain] INFO  [,,] o.h.annotations.common.Version - HCANN000001: Hibernate Commons Annotations {5.1.0.Final}
2019-11-21 11:45:10,253 [restartedMain] INFO  [,,] org.hibernate.dialect.Dialect - HHH000400: Using dialect: org.hibernate.dialect.Oracle10gDialect
2019-11-21 11:45:15,084 [restartedMain] INFO  [,,] o.h.e.t.j.p.i.JtaPlatformInitiator - HHH000490: Using JtaPlatform implementation: [org.hibernate.engine.transaction.jta.platform.internal.NoJtaPlatform]
2019-11-21 11:45:15,092 [restartedMain] INFO  [,,] o.s.o.j.LocalContainerEntityManagerFactoryBean - Initialized JPA EntityManagerFactory for persistence unit 'default'
2019-11-21 11:45:15,134 [restartedMain] WARN  [,,] o.s.b.d.a.OptionalLiveReloadServer - Unable to start LiveReload server
2019-11-21 11:45:15,609 [restartedMain] WARN  [,,] o.s.b.a.o.j.JpaBaseConfiguration$JpaWebConfiguration$JpaWebMvcConfiguration - spring.jpa.open-in-view is enabled by default. Therefore, database queries may be performed during view rendering. Explicitly configure spring.jpa.open-in-view to disable this warning
2019-11-21 11:45:15,713 [restartedMain] INFO  [,,] o.s.b.a.e.web.EndpointLinksResolver - Exposing 2 endpoint(s) beneath base path '/actuator'
2019-11-21 11:45:15,773 [restartedMain] INFO  [,,] s.d.s.w.PropertySourcedRequestMappingHandlerMapping - Mapped URL path [/v2/api-docs] onto method [public org.springframework.http.ResponseEntity<springfox.documentation.spring.web.json.Json> springfox.documentation.swagger2.web.Swagger2Controller.getDocumentation(java.lang.String,javax.servlet.http.HttpServletRequest)]
2019-11-21 11:45:15,915 [restartedMain] INFO  [,,] o.s.s.c.ThreadPoolTaskExecutor - Initializing ExecutorService 'applicationTaskExecutor'
2019-11-21 11:45:15,990 [restartedMain] WARN  [,,] o.s.b.a.t.ThymeleafAutoConfiguration$DefaultTemplateResolverConfiguration - Cannot find template location: classpath:/templates/ (please add some templates or check your Thymeleaf configuration)
2019-11-21 11:45:16,083 [restartedMain] INFO  [,,] o.s.s.c.ThreadPoolTaskScheduler - Initializing ExecutorService
2019-11-21 11:45:16,211 [restartedMain] WARN  [,,] o.s.b.a.f.FreeMarkerAutoConfiguration - Cannot find template location(s): [classpath:/templates/] (please add some templates, check your FreeMarker configuration, or set spring.freemarker.checkTemplateLocation=false)
2019-11-21 11:45:16,366 [restartedMain] INFO  [,,] s.d.s.w.p.DocumentationPluginsBootstrapper - Context refreshed
2019-11-21 11:45:16,380 [restartedMain] INFO  [,,] s.d.s.w.p.DocumentationPluginsBootstrapper - Found 1 custom documentation plugin(s)
2019-11-21 11:45:16,401 [restartedMain] INFO  [,,] s.d.s.w.s.ApiListingReferenceScanner - Scanning for api listing references
2019-11-21 11:45:16,490 [restartedMain] INFO  [,,] o.a.coyote.http11.Http11NioProtocol - Starting ProtocolHandler ["http-nio-8045"]
2019-11-21 11:45:16,509 [restartedMain] INFO  [,,] o.s.b.w.e.tomcat.TomcatWebServer - Tomcat started on port(s): 8045 (http) with context path ''
2019-11-21 11:45:16,512 [restartedMain] INFO  [,,] com.xiao.emsoft.DemoApplication - Started DemoApplication in 34.395 seconds (JVM running for 40.076)
2019-11-21 11:45:16,644 [RMI TCP Connection(3)-192.168.31.108] INFO  [,,] o.a.c.c.C.[Tomcat].[localhost].[/] - Initializing Spring DispatcherServlet 'dispatcherServlet'
2019-11-21 11:45:16,645 [RMI TCP Connection(3)-192.168.31.108] INFO  [,,] o.s.web.servlet.DispatcherServlet - Initializing Servlet 'dispatcherServlet'
2019-11-21 11:45:16,655 [RMI TCP Connection(3)-192.168.31.108] INFO  [,,] o.s.web.servlet.DispatcherServlet - Completed initialization in 10 ms
2019-11-21 11:45:45,341 [registrationTask1] WARN  [,,] d.c.b.a.c.r.ApplicationRegistrator - Failed to register application as Application(name=mytest, managementUrl=http://192.168.31.108:8045/actuator, healthUrl=http://192.168.31.108:8045/actuator/health, serviceUrl=http://192.168.31.108:8045/) at spring-boot-admin ([http://localhost:8045/instances]): 404 null. Further attempts are logged on DEBUG level
2019-11-21 11:49:26,156 [http-nio-8045-exec-1] ERROR [,,] o.a.c.c.C.[.[.[.[dispatcherServlet] - Servlet.service() for servlet [dispatcherServlet] in context with path [] threw exception [Request processing failed; nested exception is org.springframework.orm.jpa.JpaSystemException: Unknown integral data type for ids : java.lang.String; nested exception is org.hibernate.id.IdentifierGenerationException: Unknown integral data type for ids : java.lang.String] with root cause
org.hibernate.id.IdentifierGenerationException: Unknown integral data type for ids : java.lang.String
	at org.hibernate.id.IdentifierGeneratorHelper.getIntegralDataTypeHolder(IdentifierGeneratorHelper.java:224)
	at org.hibernate.id.enhanced.SequenceStructure$1.getNextValue(SequenceStructure.java:98)
	at org.hibernate.id.enhanced.NoopOptimizer.generate(NoopOptimizer.java:40)
	at org.hibernate.id.enhanced.SequenceStyleGenerator.generate(SequenceStyleGenerator.java:520)
	at org.hibernate.event.internal.AbstractSaveEventListener.saveWithGeneratedId(AbstractSaveEventListener.java:120)
	at org.hibernate.event.internal.DefaultMergeEventListener.saveTransientEntity(DefaultMergeEventListener.java:271)
	at org.hibernate.event.internal.DefaultMergeEventListener.entityIsTransient(DefaultMergeEventListener.java:243)
	at org.hibernate.event.internal.DefaultMergeEventListener.entityIsDetached(DefaultMergeEventListener.java:318)
	at org.hibernate.event.internal.DefaultMergeEventListener.onMerge(DefaultMergeEventListener.java:172)
	at org.hibernate.event.internal.DefaultMergeEventListener.onMerge(DefaultMergeEventListener.java:70)
	at org.hibernate.event.service.internal.EventListenerGroupImpl.fireEventOnEachListener(EventListenerGroupImpl.java:108)
	at org.hibernate.internal.SessionImpl.fireMerge(SessionImpl.java:776)
	at org.hibernate.internal.SessionImpl.merge(SessionImpl.java:763)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.springframework.orm.jpa.ExtendedEntityManagerCreator$ExtendedEntityManagerInvocationHandler.invoke(ExtendedEntityManagerCreator.java:350)
	at com.sun.proxy.$Proxy137.merge(Unknown Source)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.springframework.orm.jpa.SharedEntityManagerCreator$SharedEntityManagerInvocationHandler.invoke(SharedEntityManagerCreator.java:308)
	at com.sun.proxy.$Proxy137.merge(Unknown Source)
	at org.springframework.data.jpa.repository.support.SimpleJpaRepository.save(SimpleJpaRepository.java:499)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.springframework.data.repository.core.support.RepositoryComposition$RepositoryFragments.invoke(RepositoryComposition.java:359)
	at org.springframework.data.repository.core.support.RepositoryComposition.invoke(RepositoryComposition.java:200)
	at org.springframework.data.repository.core.support.RepositoryFactorySupport$ImplementationMethodExecutionInterceptor.invoke(RepositoryFactorySupport.java:644)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
	at org.springframework.data.repository.core.support.RepositoryFactorySupport$QueryExecutorMethodInterceptor.doInvoke(RepositoryFactorySupport.java:608)
	at org.springframework.data.repository.core.support.RepositoryFactorySupport$QueryExecutorMethodInterceptor.lambda$invoke$3(RepositoryFactorySupport.java:595)
	at org.springframework.data.repository.core.support.RepositoryFactorySupport$QueryExecutorMethodInterceptor.invoke(RepositoryFactorySupport.java:595)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
	at org.springframework.data.projection.DefaultMethodInvokingMethodInterceptor.invoke(DefaultMethodInvokingMethodInterceptor.java:59)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:295)
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:98)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
	at org.springframework.dao.support.PersistenceExceptionTranslationInterceptor.invoke(PersistenceExceptionTranslationInterceptor.java:139)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
	at org.springframework.data.jpa.repository.support.CrudMethodMetadataPostProcessor$CrudMethodMetadataPopulatingMethodInterceptor.invoke(CrudMethodMetadataPostProcessor.java:138)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
	at org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:93)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
	at org.springframework.data.repository.core.support.SurroundingTransactionDetectorMethodInterceptor.invoke(SurroundingTransactionDetectorMethodInterceptor.java:61)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
	at org.springframework.aop.framework.JdkDynamicAopProxy.invoke(JdkDynamicAopProxy.java:212)
	at com.sun.proxy.$Proxy139.save(Unknown Source)
	at com.xiao.emsoft.infrastructure.jpa.impl.MyTestResitoryjpa.test(MyTestResitoryjpa.java:34)
	at com.xiao.emsoft.infrastructure.jpa.impl.MyTestResitoryjpa$$FastClassBySpringCGLIB$$e6afc608.invoke(<generated>)
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:218)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:749)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:163)
	at org.springframework.dao.support.PersistenceExceptionTranslationInterceptor.invoke(PersistenceExceptionTranslationInterceptor.java:139)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:688)
	at com.xiao.emsoft.infrastructure.jpa.impl.MyTestResitoryjpa$$EnhancerBySpringCGLIB$$e034d0cc.test(<generated>)
	at com.xiao.emsoft.domain.service.MyTestService.test(MyTestService.java:23)
	at com.xiao.emsoft.controller.impl.HellowService.sayhellow(HellowService.java:24)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.springframework.web.method.support.InvocableHandlerMethod.doInvoke(InvocableHandlerMethod.java:190)
	at org.springframework.web.method.support.InvocableHandlerMethod.invokeForRequest(InvocableHandlerMethod.java:138)
	at org.springframework.web.servlet.mvc.method.annotation.ServletInvocableHandlerMethod.invokeAndHandle(ServletInvocableHandlerMethod.java:104)
	at org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerAdapter.invokeHandlerMethod(RequestMappingHandlerAdapter.java:892)
	at org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerAdapter.handleInternal(RequestMappingHandlerAdapter.java:797)
	at org.springframework.web.servlet.mvc.method.AbstractHandlerMethodAdapter.handle(AbstractHandlerMethodAdapter.java:87)
	at org.springframework.web.servlet.DispatcherServlet.doDispatch(DispatcherServlet.java:1039)
	at org.springframework.web.servlet.DispatcherServlet.doService(DispatcherServlet.java:942)
	at org.springframework.web.servlet.FrameworkServlet.processRequest(FrameworkServlet.java:1005)
	at org.springframework.web.servlet.FrameworkServlet.doGet(FrameworkServlet.java:897)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:634)
	at org.springframework.web.servlet.FrameworkServlet.service(FrameworkServlet.java:882)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:741)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at org.springframework.boot.actuate.web.trace.servlet.HttpTraceFilter.doFilterInternal(HttpTraceFilter.java:90)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:107)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at org.springframework.web.filter.RequestContextFilter.doFilterInternal(RequestContextFilter.java:99)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:107)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at org.springframework.web.filter.FormContentFilter.doFilterInternal(FormContentFilter.java:92)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:107)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at org.springframework.web.filter.HiddenHttpMethodFilter.doFilterInternal(HiddenHttpMethodFilter.java:93)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:107)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at org.springframework.boot.actuate.metrics.web.servlet.WebMvcMetricsFilter.filterAndRecordMetrics(WebMvcMetricsFilter.java:117)
	at org.springframework.boot.actuate.metrics.web.servlet.WebMvcMetricsFilter.doFilterInternal(WebMvcMetricsFilter.java:106)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:107)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at org.springframework.web.filter.CharacterEncodingFilter.doFilterInternal(CharacterEncodingFilter.java:200)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:107)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:200)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:96)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:490)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:139)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:74)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:343)
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:408)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:66)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:836)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1747)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
2019-11-21 11:50:12,000 [Thread-79] INFO  [,,] o.s.s.c.ThreadPoolTaskScheduler - Shutting down ExecutorService
2019-11-21 11:50:12,001 [Thread-79] INFO  [,,] o.s.s.c.ThreadPoolTaskExecutor - Shutting down ExecutorService 'applicationTaskExecutor'
2019-11-21 11:50:12,003 [Thread-79] INFO  [,,] o.s.o.j.LocalContainerEntityManagerFactoryBean - Closing JPA EntityManagerFactory for persistence unit 'default'
2019-11-21 11:50:12,025 [Thread-79] INFO  [,,] com.zaxxer.hikari.HikariDataSource - HikariPool-1 - Shutdown initiated...
2019-11-21 11:50:12,082 [Thread-79] INFO  [,,] com.zaxxer.hikari.HikariDataSource - HikariPool-1 - Shutdown completed.
